--- stratagems/stratagems.ini
+++ stratagems/stratagems.ini
@@ -8,7 +8,7 @@ bonus_spell_scrolls=1
 circle_of_bones_duration=1
 clone_slots=1
 death_spell_summons=1
-death_spell_summons_iwd=1
+death_spell_summons_iwd=0
 eldath_rename=1
 elemental_summoning=1
 energy_drain=1
@@ -107,7 +107,7 @@ Sendai_Component_Is_Sendai_Only = 0
 force_old_assassin_spawn = 0
 no_spirit_troll_magic = 0
 no_swift_shapeshift = 1
-potion_version=6
+potion_version=1
 dispel_arrow_maximum=5
 do_not_lose_soul=0
 keep_cause_wounds_alignment_restrictions = 0
@@ -119,12 +119,12 @@ teleport_maze_number=0
 hitpoint_percentage=0
 no_nabassu_color_change=0
 no_kahrk_treasure=0
-resurrection_requires_diamond=1
-resurrection_rod_cost_per_charge=5000
+resurrection_requires_diamond=0
+resurrection_rod_cost_per_charge=2000
 resurrection_spell_cost_uptick=1000
-imprisonment_end_duration=3600
+imprisonment_end_duration=1800
 slayer_change_on_imprisonment_end=1
-resurrection_rod_max_charges=3
+resurrection_rod_max_charges=10
 
 
 
