diff --git a/A7-ChaosSorcerer/languages/english/kit.tra b/A7-ChaosSorcerer/languages/english/kit.tra
index d4b213d..a5a4014 100644
--- A7-ChaosSorcerer/languages/english/kit.tra
+++ A7-ChaosSorcerer/languages/english/kit.tra
@@ -25,7 +25,7 @@ Advantages:
 – 24th level: +5% Magic Resistance.
 
 Disadvantages:
-– Incurs a -2 penalty to Strength and Constitution.
+– Incurs a -1 penalty to Strength and Constitution.
 – Alignment restricted to chaotic good, chaotic neutral and chaotic evil.
 – May cast fewer spells per level per day.
 – Upon casting a spell, there is a 5% chance of incurring a wild surge.
diff --git a/A7-ChaosSorcerer/languages/french/kit.tra b/A7-ChaosSorcerer/languages/french/kit.tra
index b8259f5..431d779 100644
--- A7-ChaosSorcerer/languages/french/kit.tra
+++ A7-ChaosSorcerer/languages/french/kit.tra
@@ -25,7 +25,7 @@ Avantages:
 – Niveau 24 : résistance à la magie : +5%.
 
 Inconvénients :
-– Subit une pénalité de -2 à la force et à la constitution.
+– Subit une pénalité de -1 à la force et à la constitution.
 – Alignement limité à chaotique bon, chaotique neutre et chaotique mauvais.
 – Peut lancer moins de sorts par jour.
 – Chaque fois qu'il lance un sort, 5% de chance de subir un hiatus entropique.
diff --git a/A7-ChaosSorcerer/lib/kit.tph b/A7-ChaosSorcerer/lib/kit.tph
index afc9541..82cec52 100644
--- A7-ChaosSorcerer/lib/kit.tph
+++ A7-ChaosSorcerer/lib/kit.tph
@@ -50,7 +50,7 @@ STR_VAR
   clasweap  = ~1 1 1 1 1 0 0 0~
   weapprof  = ~0 1 0 0 1 0 0 1 0 0 0 0 0 0 0 1 0 0 0 0 0 0 1 0 0 0 1 1~
   abclasrq  = ~0 12 0 9 0 9~
-  abclsmod  = ~-2 0 -2 0 0 0~
+  abclsmod  = ~-1 0 -1 0 0 0~
   abdcdsrq  = ~0 17 0 17 0 17~
   abdcscrq  = ~0 15 0 15 0 15~
   alignmnt  = ~0 0 0 0 0 0 1 1 1~
