--- Aura_BG1/Setup-Aura_BG1.tp2
+++ Aura_BG1/Setup-Aura_BG1.tp2
@@ -40,22 +40,9 @@ THEN BEGIN
 INCLUDE ~%MOD_FOLDER%/lib/aura_sod.tpa~
 END
 
-PRINT ~Choose a portrait for Aura (check mod folder for portraits):~
 
-PRINT ~Please choose one of the following:
-[1] BG1 Default by Artemius_I/AionZ
-[2] BG2 Default by Artemius_I/AionZ
-[3] Alternate 1 by Omar Diaz
-[4] Alternate 2 by Pantalion
-[5] Alternate 3 (edited NWN)
-[6] First BG1 Default by Lava Del'Vortel
-[7] Second BG1 Default by Artemius_I/AionZ
-[8] Original BG2 Default by Artemius_I/AionZ~
+OUTER_SPRINT ~portrait~ 2
 
-OUTER_SPRINT ~portrait~ ~placeholder_value~
-OUTER_WHILE (!(IS_AN_INT ~portrait~) OR (~portrait~ > 0x7) OR (~portrait~ < 0x1)) BEGIN
-  ACTION_READLN ~portrait~
-END
       ACTION_IF ("portrait" = 8) THEN BEGIN
 		COPY ~%MOD_FOLDER%/alt-portrait/originalbg2/AuraL.bmp~ ~override/AuraL.bmp~
 		COPY ~%MOD_FOLDER%/alt-portrait/originalbg2/AuraM.bmp~ ~override/AuraM.bmp~
--- Aura_BG1/lib/aura_kit.tpa
+++ Aura_BG1/lib/aura_kit.tpa
@@ -2,18 +2,8 @@
 //////////////////////////////////// KIT //////////////////////////////////////
 ///////////////////////////////////////////////////////////////////////////////
 
-PRINT ~Choose a class for Aura:~
+OUTER_SPRINT ~auraclass~ 1
 
-PRINT ~Please choose one of the following:
-[1] Artificer (Thief)
-[2] Illusionist / Artificer (Mage / Thief)
-[3] Priestess / Artificer (Cleric / Thief)~
-
-OUTER_SPRINT ~auraclass~ ~placeholder_value~
-OUTER_WHILE (!(IS_AN_INT ~auraclass~) OR (~auraclass~ > 0x5) OR (~auraclass~ < 0x1)) BEGIN
-  PRINT ~Please select 1, 2, or 3 and press Enter.~
-  ACTION_READLN ~auraclass~
-END
       ACTION_IF ("auraclass" = 1) THEN BEGIN
 ADD_KIT ~C0AURA~
  // appended to CLASWEAP.2da
