--- c#brage/baf/C#BrageD.baf
+++ c#brage/baf/C#BrageD.baf
@@ -17,7 +17,7 @@ IF
 THEN
 	RESPONSE #100
 		SetGlobal("C#BE_BrageTalksBrooch","GLOBAL",1)
-		RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",3200)
+		RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",320)
 		StartDialogueNoSet(Player1)
 		
 END
--- c#brage/baf/c#brage1.baf
+++ c#brage/baf/c#brage1.baf
@@ -371,7 +371,7 @@ IF
 	Global("C#BE_BrageCaravan","GLOBAL",1) 
 THEN
 	RESPONSE #100
-		RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",3200)
+		RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",320)
 		StartDialogueNoSet(Player1)
 		
 END
@@ -415,7 +415,7 @@ IF
 	Global("C#BE_KnowOfBhaal","GLOBAL",2) 
 THEN
 	RESPONSE #100
-		RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",3200)
+		RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",320)
 		StartDialogueNoSet(Player1)
 		
 END
@@ -1368,7 +1368,7 @@ IF
 	Global("C#BE_BrageTalkPoppies","GLOBAL",1) 
 THEN
 	RESPONSE #100
-		RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",3200)
+		RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",320)
 		StartDialogueNoSet(Player1)
 		
 END
@@ -1403,7 +1403,7 @@ IF
 	Global("C#BE_GraveIncident","GLOBAL",1) 
 THEN
 	RESPONSE #100
-		RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",3200)
+		RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",320)
 		StartDialogueNoSet(Player1)
 		
 END
@@ -1440,7 +1440,7 @@ IF
 	Global("C#BE_GraveIncident","GLOBAL",3) 
 THEN
 	RESPONSE #100
-		RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",3200)
+		RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",320)
 		StartDialogueNoSet(Player1)
 		
 END
@@ -1527,7 +1527,7 @@ IF
 		Global("C#BE_GraveIncident","GLOBAL",10) 
 THEN
 	RESPONSE #100
-		RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",3200)
+		RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",320)
 		StartDialogueNoSet(Player1)		
 END
 
@@ -1560,7 +1560,7 @@ IF
 	Global("C#BE_GraveIncident","GLOBAL",13) 
 THEN
 	RESPONSE #100
-		RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",3200)
+		RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",320)
 		StartDialogueNoSet(Player1)		
 END
 
@@ -1593,7 +1593,7 @@ IF
 	Global("C#BE_GraveIncident","GLOBAL",16) 
 THEN
 	RESPONSE #100
-		RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",3200)
+		RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",320)
 		StartDialogueNoSet(Player1)
 		
 END
@@ -1653,7 +1653,7 @@ IF
 		Global("C#BE_BrageTalks","GLOBAL",22)   
 THEN
 	RESPONSE #100
-		RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",3200)
+		RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",320)
 		StartDialogueNoSet(Player1)
 		
 END
--- c#brage/baf/c#brage2.baf
+++ c#brage/baf/c#brage2.baf
@@ -44,7 +44,7 @@ IF
 	!Global("C#BE_LilarcorEquipped","GLOBAL",1)
 THEN
 	RESPONSE #100
-		RealSetGlobalTimer("C#BE_LilarcorEquippedTimer","GLOBAL",3600)
+		RealSetGlobalTimer("C#BE_LilarcorEquippedTimer","GLOBAL",360)
 		SetGlobal("C#BE_LilarcorEquipped","GLOBAL",1)
 END
 
--- c#brage/crossmod/brage_bgqe_babysitting.baf
+++ c#brage/crossmod/brage_bgqe_babysitting.baf
@@ -10,7 +10,7 @@ THEN
 	RESPONSE #100
 		SetGlobal("C#BE_BGQETimerSet","LOCALS",1)
 		SetGlobal("C#BE_Q03Babysitting","GLOBAL",1)
-		RealSetGlobalTimer("C#BE_BGQETimer","GLOBAL",1000)
+		RealSetGlobalTimer("C#BE_BGQETimer","GLOBAL",100)
 END
 
 
--- c#brage/crossmod/brage_bgqe_fallenpaladin.baf
+++ c#brage/crossmod/brage_bgqe_fallenpaladin.baf
@@ -11,7 +11,7 @@ THEN
 	RESPONSE #100
 		SetGlobal("C#BE_C#Q05001_15","GLOBAL",2)
 		SetGlobal("C#BE_BGQETimerSet","LOCALS",1)
-		RealSetGlobalTimer("C#BE_BGQETimer","GLOBAL",1000)
+		RealSetGlobalTimer("C#BE_BGQETimer","GLOBAL",100)
 END
 
 /* activate */
--- c#brage/crossmod/brage_bgqe_nashkelmonster.baf
+++ c#brage/crossmod/brage_bgqe_nashkelmonster.baf
@@ -29,7 +29,7 @@ THEN
 	RESPONSE #100
 		SetGlobal("C#BE_BGQETimerSet","LOCALS",1)
 		SetGlobal("C#BE_Q04NashkelMonster","GLOBAL",1)
-		RealSetGlobalTimer("C#BE_BGQETimer","GLOBAL",1000)
+		RealSetGlobalTimer("C#BE_BGQETimer","GLOBAL",100)
 END
 
 /* activate */
--- c#brage/crossmod/brage_bgqe_noober.baf
+++ c#brage/crossmod/brage_bgqe_noober.baf
@@ -10,7 +10,7 @@ THEN
 	RESPONSE #100
 		SetGlobal("C#BE_BGQETimerSet","LOCALS",1)
 		SetGlobal("C#BE_BGQENoober","GLOBAL",1)
-		RealSetGlobalTimer("C#BE_BGQETimer","GLOBAL",1000)
+		RealSetGlobalTimer("C#BE_BGQETimer","GLOBAL",100)
 END
 
 /* activate */
--- c#brage/dialogue/bg1.d
+++ c#brage/dialogue/bg1.d
@@ -1107,7 +1107,7 @@ END
 IF ~~ THEN friendtalk7_03
 SAY @234
 IF ~~ THEN DO ~IncrementGlobal("C#BE_BrageTalks","GLOBAL",1)
-RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",800)~ EXIT //shorter timer
+RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",100)~ EXIT //shorter timer
 END
 
 IF ~~ THEN friendtalk7_04
--- c#brage/dialogue/joining_bg1.d
+++ c#brage/dialogue/joining_bg1.d
@@ -74,6 +74,6 @@ IF ~Global("C#BE_SpawnBrage","GLOBAL",7)~ THEN join
 SAY @18 
 IF ~~ THEN DO ~SetGlobal("C#BE_SpawnBrage","GLOBAL",8) 
 SetGlobal("Kickout","LOCALS",0)
-SetGlobal("C#BE_BrageTalks","GLOBAL",1) RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",800) JoinParty()~ EXIT
+SetGlobal("C#BE_BrageTalks","GLOBAL",1) RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",200) JoinParty()~ EXIT
 END
 
--- c#brage/dialogue/joining_bg2.d
+++ c#brage/dialogue/joining_bg2.d
@@ -80,7 +80,7 @@ IF ~~ THEN C#Brage2 joinbg2_06
 == C#Brage2 IF ~Dead("UnseeingEye")~ THEN ~You already clinsed the city of the "Unseeing Eye" cult, as I see. This leaves me with no more obligations to the temples.~
 == C#Brage2 IF ~Exists("OISIG")~ THEN ~Please, take me to High Watcher Oisig, so I can tell him my farewell.~
 END
-IF ~~ THEN DO ~SetGlobal("C#BE_SpawnBrage","GLOBAL",22) SetGlobal("C#BE_BrageTalksBG2","GLOBAL",1) RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",1600) JoinParty()~ EXIT
+IF ~~ THEN DO ~SetGlobal("C#BE_SpawnBrage","GLOBAL",22) SetGlobal("C#BE_BrageTalksBG2","GLOBAL",1) RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",300) JoinParty()~ EXIT
 
 
 CHAIN
--- c#brage/dialogue/joining_tob.d
+++ c#brage/dialogue/joining_tob.d
@@ -10,7 +10,7 @@ END
 
 IF ~~ THEN joinbg2_01
 SAY @25 /* ~Gladly.~ */
-IF ~~ THEN DO ~SetGlobal("C#BrageSummoned","GLOBAL",4) SetGlobal("C#BE_BrageTalksToB","GLOBAL",1) RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",800) JoinParty()~ EXIT
+IF ~~ THEN DO ~SetGlobal("C#BrageSummoned","GLOBAL",4) SetGlobal("C#BE_BrageTalksToB","GLOBAL",1) RealSetGlobalTimer("C#BE_BrageDialogueTimer","GLOBAL",300) JoinParty()~ EXIT
 END
 
 IF ~~ THEN joinbg2_02
