diff --git a/HerThiMoney.tp2 b/HerThiMoney.tp2
index 85789d7..0ce4fe4 100644
--- a/HerThiMoney.tp2
+++ b/HerThiMoney.tp2
@@ -409,7 +409,6 @@ ACTION_IF EE_GAME BEGIN
   ADD_JOURNAL @1025 @1026 @1038 @1039 USING ~%MOD_FOLDER%/tra/%LANGUAGE%/setup.tra~ 
 END 
 
-AT_INTERACTIVE_EXIT ~VIEW HerThiMoney/readme/%LANGUAGE%/readme.pdf~
 
 
 
@@ -1158,7 +1157,6 @@ COPY "%MOD_FOLDER%/cre/bdoffscr.cre" override
 COPY "%MOD_FOLDER%/spl/bdoffscr.spl" override
 END
 
-AT_INTERACTIVE_EXIT ~VIEW HerThiMoney/readme/%LANGUAGE%/readme.pdf~
 
 //
 //
@@ -1603,7 +1601,6 @@ COPY_EXISTING ~ar0307.are~ ~override~
 	END
 	BUT_ONLY_IF_IT_CHANGES
 
-AT_INTERACTIVE_EXIT ~VIEW HerThiMoney/readme/%LANGUAGE%/readme.pdf~
 
 
 //
@@ -1818,7 +1815,6 @@ END
 COPY_EXISTING ~WAUKSQU.cre~ ~override\WAUKSQU.cre~
 WRITE_ASCII 0x280 ~ANbelka~ (32)
 
-AT_INTERACTIVE_EXIT ~VIEW HerThiMoney/readme/%LANGUAGE%/readme.pdf~
 
 
 
@@ -1840,4 +1836,4 @@ BEGIN @157 DESIGNATED 40
 // Патчим файл player1, чтобы добавить диалоги всем NPC после превращения игрока в Убийцу (ранее говорил только кто-то один), подключаем функцию
 LAM AN_PLAYER1DLG_PATCH
 
-COMPILE ~HerThiMoney/dlg/ANtwick.d~
\ No newline at end of file
+COMPILE ~HerThiMoney/dlg/ANtwick.d~
