--- HerThiMoney/HerThiMoney.tp2     2021-08-19 15:56:16.620622627 +0200
+++ HerThiMoney/HerThiMoney.tp2     2021-08-19 16:01:12.194389680 +0200
@@ -409,7 +409,6 @@ ACTION_IF EE_GAME BEGIN
   ADD_JOURNAL @1025 @1026 @1038 @1039 USING ~%MOD_FOLDER%/tra/%LANGUAGE%/setup.tra~ 
 END 
 
-AT_INTERACTIVE_EXIT ~VIEW HerThiMoney/readme/%LANGUAGE%/readme.pdf~
 
 
 
@@ -1158,7 +1157,6 @@ COPY "%MOD_FOLDER%/cre/bdoffscr.cre" override
 COPY "%MOD_FOLDER%/spl/bdoffscr.spl" override
 END
 
-AT_INTERACTIVE_EXIT ~VIEW HerThiMoney/readme/%LANGUAGE%/readme.pdf~
 
 //
 //
@@ -1603,7 +1601,6 @@ COPY_EXISTING ~ar0307.are~ ~override~
 	END
 	BUT_ONLY_IF_IT_CHANGES
 
-AT_INTERACTIVE_EXIT ~VIEW HerThiMoney/readme/%LANGUAGE%/readme.pdf~
 
 
 //
@@ -1818,7 +1815,6 @@ END
 COPY_EXISTING ~WAUKSQU.cre~ ~override\WAUKSQU.cre~
 WRITE_ASCII 0x280 ~ANbelka~ (32)
 
-AT_INTERACTIVE_EXIT ~VIEW HerThiMoney/readme/%LANGUAGE%/readme.pdf~
 
 
 