--- mih_eq/bg2/firkraags_lair/firkraag_battle.tpa
+++ mih_eq/bg2/firkraags_lair/firkraag_battle.tpa
@@ -24,14 +24,14 @@ BEGIN
 
   LAUNCH_ACTION_FUNCTION install
     STR_VAR
-    file	= "mh#rdmin.baf"
+    file	= "mh#rdrag.baf"
     location	= "scripts"
   END
 
   LAUNCH_ACTION_FUNCTION edit_creature
     STR_VAR
     creature	= "firkra02"
-    editstring	= ~insert_script_high=>"mh#rdmin" make_legendary=>"%firkraag_legendary_rank%" enforce_xp=>10~
+    editstring	= ~insert_script_high=>"mh#rdrag" make_legendary=>"%firkraag_legendary_rank%" enforce_xp=>10~
   END
 END	// firkraag_battle
 
--- mih_eq/bg2/suldanessenar/black_dragon_battle.tpa
+++ mih_eq/bg2/suldanessenar/black_dragon_battle.tpa
@@ -36,7 +36,7 @@ BEGIN
   LAUNCH_ACTION_FUNCTION extend_area_script
     STR_VAR
     area	= "ar2807"
-    bottom	= "spawn_black_dragon_minions"
+    bottom	= "mh#bdrag"
     location	= "scripts"
   END
 END	// black_dragon_battle
--- mih_eq/bg2/temple_ruins/shadow_dragon_battle.tpa
+++ mih_eq/bg2/temple_ruins/shadow_dragon_battle.tpa
@@ -24,14 +24,14 @@ BEGIN
 
   LAUNCH_ACTION_FUNCTION install
     STR_VAR
-    file	= "mh#sdmin.baf"
+    file	= "mh#sdrag.baf"
     location	= "scripts"
   END
 
   LAUNCH_ACTION_FUNCTION edit_creature
     STR_VAR
     creature	= "dragshad"
-    editstring	= ~insert_script_high=>"mh#sdmin" make_legendary=>"%thaxll_legendary_rank%" enforce_xp=>10~
+    editstring	= ~insert_script_high=>"mh#sdrag" make_legendary=>"%thaxll_legendary_rank%" enforce_xp=>10~
   END
 END	// shadow_dragon_battle
 
--- mih_eq/mih_eq.ini
+++ mih_eq/mih_eq.ini
@@ -10,19 +10,19 @@ infinity_animations=0
 implement_lore_changes=1
 
 [revised_dragons]
-dragon_hp_per_hd=12
+dragon_hp_per_hd=7
 
 [revised_liches]
-lich_spell_level_immunity=3
-lich_powers_save_penalty=6
-demilich_howl_save_penalty=4
-demilich_curse_debuff=8
+lich_spell_level_immunity=2
+lich_powers_save_penalty=3
+demilich_howl_save_penalty=2
+demilich_curse_debuff=4
 
 [revised_lycanthropes]
 lycanthrope_hd_bonus=2
 
 [revised_magical_beasts]
-dread_wolf_disease_duration=2400
+dread_wolf_disease_duration=1200
 relocate_winter_wolves=1
 wyvern_hd_bonus=4
 wyvern_ac_bonus=2
@@ -32,68 +32,68 @@ spider_hd_bonus=3
 spider_ac_bonus=4
 
 [ankheg_pit]
-ankheg_wave_count=10
-ankheg_wave_interval=ONE_TURN
+ankheg_wave_count=6
+ankheg_wave_interval=THREE_TURNS
 
 [spider_plague]
-spider_wave_count=20
-spider_wave_interval=TWO_TURNS
+spider_wave_count=10
+spider_wave_interval=FIVE_TURNS
 
 [sod_enemy_groups]
-sod_group_size=4
+sod_group_size=3
 
 [korlasz_tomb]
-korlasz_tomb_level_bonus=5
+korlasz_tomb_level_bonus=4
 
 [zavians_vision_quest]
 spectacle_charges=9
 
 [firkraags_lair]
-firkraag_legendary_rank=scion
-firkraag_minion_count=10
-firkraag_minion_interval=THREE_ROUNDS
+firkraag_legendary_rank=lesser_scion
+firkraag_minion_count=4
+firkraag_minion_interval=FIVE_ROUNDS
 
 [underdark]
 kuotoa_level_bonus=6
-deirex_minion_count=10
-deirex_minion_interval=ONE_ROUND
+deirex_minion_count=4
+deirex_minion_interval=FIVE_ROUNDS
 
 [temple_ruins]
-thaxll_legendary_rank=scion
-thaxll_minion_count=10
-thaxll_minion_interval=THREE_ROUNDS
+thaxll_legendary_rank=lesser_scion
+thaxll_minion_count=4
+thaxll_minion_interval=FIVE_ROUNDS
 
 [suldanessenar]
-nizid_legendary_rank=scion
-nizid_minion_count=10
-nizid_minion_interval=THREE_ROUNDS
+nizid_legendary_rank=lesser_scion
+nizid_minion_count=4
+nizid_minion_interval=FIVE_ROUNDS
 
 [watchers_keep]
-saladrex_legendary_rank=elder
-saladrex_minion_count=10
-saladrex_minion_interval=THREE_ROUNDS
-trial_dragon_legendary_rank=elder
-trial_dragon_minion_count=4
-trial_dragon_minion_interval=THREE_ROUNDS
-orc_horde_count=20
-orc_horde_duration=THREE_TURNS
-orc_horde_level_bonus=10
+saladrex_legendary_rank=scion
+saladrex_minion_count=4
+saladrex_minion_interval=FIVE_ROUNDS
+trial_dragon_legendary_rank=scion
+trial_dragon_minion_count=3
+trial_dragon_minion_interval=FIVE_ROUNDS
+orc_horde_count=15
+orc_horde_duration=FIVE_TURNS
+orc_horde_level_bonus=8
 
 [pocket_plane]
-sarevok_party_level_bonus=10
+sarevok_party_level_bonus=7
 
 [saradush]
-barracks_battle_level_bonus=8
+barracks_battle_level_bonus=5
 prison_vampires_legendary_rank=lesser_scion
 
 [forest_of_mir]
-master_wraith_minion_count=10
-master_wraith_minion_interval=THREE_ROUNDS
+master_wraith_minion_count=6
+master_wraith_minion_interval=FIVE_ROUNDS
 
 [yaga-shuras_army]
 yaga_shura_army_level_bonus=3
-yaga_shura_army_count=20
-yaga_shura_army_interval=THREE_ROUNDS
+yaga_shura_army_count=15
+yaga_shura_army_interval=FIVE_ROUNDS
 
 [metadata]
 Name=Made in Heaven: Encounters & Quests
