--- mih_ip/lib/lib_ini.tpa        2021-08-20 09:43:06.927530954 +0200
+++ mih_ip/lib/lib_ini.tpa        2021-08-20 09:43:19.475925300 +0200
@@ -43,7 +43,7 @@ BEGIN
         PATCH_IF !VARIABLE_IS_SET $SFO_reserved_ini_hash(~%ini%~) BEGIN
            SPRINT value 0
            PATCH_IF !silent BEGIN
-				PATCH_WARN "ini entry %ini% has been checked, but is not present in the ini file"
+				PATCH_LOG "ini entry %ini% has been checked, but is not present in the ini file"
 		   END
         END ELSE BEGIN
            SPRINT value $SFO_reserved_ini_hash(~%ini%~)
--- mih_ip/mih_ip.ini
+++ mih_ip/mih_ip.ini
@@ -13,17 +13,19 @@ max_stack_ammo=0
 max_stack_thrown=0
 max_stack_potions=0
 max_stack_jewels=0
-update_itemexcl_2da=1
+update_itemexcl_2da=2
 visible_ioun_stones=1
-brokk_buying_markup=70
-brokk_selling_markup=130
+brokk_buying_markup=80
+brokk_selling_markup=120
+phendra_buying_markup=70
+phendra_selling_markup=130
 
 [better_items]
-baldurans_cloak_magic_resistance=10
+baldurans_cloak_magic_resistance=25
 
 [extra_healing_in_bg1]
-extra_healing_minimum=2
-extra_healing_maximum=5
+extra_healing_minimum=5
+extra_healing_maximum=10
 extra_healing_shops=1
 extra_healing_durlag=1
 
