--- mih_fr/lib/lib_ini.tpa
+++ mih_fr/lib/lib_ini.tpa
@@ -43,7 +43,7 @@ BEGIN
         PATCH_IF !VARIABLE_IS_SET $SFO_reserved_ini_hash(~%ini%~) BEGIN
            SPRINT value 0
            PATCH_IF !silent BEGIN
-                               PATCH_WARN "ini entry %ini% has been checked, but is not present in the ini file"
+                               PATCH_LOG "ini entry %ini% has been checked, but is not present in the ini file"
                   END
         END ELSE BEGIN
            SPRINT value $SFO_reserved_ini_hash(~%ini%~)
