--- scales_of_balance/ini/YARAS_default.ini
+++ scales_of_balance/ini/YARAS_default.ini
@@ -45,8 +45,8 @@
 //
 //	CASTING SPEED PENALTY: 					// this is a static modifier
 //											// 	(**NOTE** the max value = 6)
-		SET leather_speed =		2
-		SET studded_speed =		2
+		SET leather_speed =		0
+		SET studded_speed =		1
 		SET chain_speed = 		3
 		SET splint_speed = 		4
 		SET plate_speed = 		5
@@ -76,9 +76,9 @@
 		  SET leather_move = 	0
 		  SET studded_move = 	0
 		  SET chain_move = 		0
-		  SET splint_move = 	0
-		  SET plate_move = 		0
-		  SET fullplate_move = 	0
+		  SET splint_move = 	1
+		  SET plate_move = 		1
+		  SET fullplate_move = 	1
 		END
 		PATCH_IF (%enchantment% >= 1) BEGIN
 		  SET leather_move = 	0
@@ -92,14 +92,6 @@
 //	DEX PENALTIES: 							// 	Dex penalties are reduced for enchanted items.
 //											// 	If you don't want that, then...
 		PATCH_IF (%enchantment% = 0) BEGIN 	// 	...make the numbers in these two blocks identical.
-		  SET leather_dex =		1
-		  SET studded_dex =		2
-		  SET chain_dex = 		3
-		  SET splint_dex = 		4
-		  SET plate_dex = 		5
-		  SET fullplate_dex = 	5
-		END
-		PATCH_IF (%enchantment% >= 1) BEGIN
 		  SET leather_dex =		0
 		  SET studded_dex =		1
 		  SET chain_dex = 		2
@@ -107,6 +99,14 @@
 		  SET plate_dex = 		4
 		  SET fullplate_dex = 	4
 		END
+		PATCH_IF (%enchantment% >= 1) BEGIN
+		  SET leather_dex =		0
+		  SET studded_dex =		0
+		  SET chain_dex = 		1
+		  SET splint_dex = 		2
+		  SET plate_dex = 		3
+		  SET fullplate_dex = 	3
+		END
 //
 //	THIEF SKILL PENALTIES:					// This affects heavy armors only; vanilla values
 //											// for leather, hide, & elven chain are unchanged
@@ -120,11 +120,11 @@
 
 		SET plate_stealth = 	75
 		SET plate_mech = 		50
-		SET plate_detect = 		30
+		SET plate_detect = 		20
 
 		SET fullplate_stealth = 90
 		SET fullplate_mech = 	60
-		SET fullplate_detect = 	40
+		SET fullplate_detect = 	20
 //											// These armors have mixed characteristics,
 //	HOW TO TREAT HIDE ARMOR:				// which you can define. E.g. hide armor: 
 //											// good DR but poor AC, DEX, and casting speed.
@@ -132,9 +132,9 @@
 		SET hide_dr = 		%chain_dr%
 		SET hide_speed =	%chain_speed%
 		SET hide_miscast =	%chain_miscast%
-		SET hide_move =		%studded_move%
+		SET hide_move =		%chain_move%
 		SET hide_weap =		%studded_weap%
-		SET hide_dex =		%chain_dex%
+		SET hide_dex =		%studded_dex%
 //
 //	HOW TO TREAT ELVEN CHAIN:				// Elven chain is as light as leather.
 //											// Or you can just set these to 0 if you want.
