--- shadowadept/lib/kit.tpa     2021-08-21 14:42:19.617390737 +0200
+++ shadowadept/lib/kit.tpa     2021-08-21 14:42:13.737203209 +0200
@@ -1,10 +1,10 @@
-PRINT ~Please choose one of the following:
-[1] Constitution drain on cast
-[2] Drain Current Hit Points on cast
-[3] Drain Maximum Hit Points on cast
-[4] No Cost~
+//PRINT ~Please choose one of the following:
+//[1] Constitution drain on cast
+//[2] Drain Current Hit Points on cast
+//[3] Drain Maximum Hit Points on cast
+//[4] No Cost~
 
-OUTER_SPRINT ~hpcost~ ~placeholder_value~
+OUTER_SPRINT ~hpcost~ 1
 OUTER_WHILE (!(IS_AN_INT ~hpcost~) OR (~hpcost~ > 0x5) OR (~hpcost~ < 0x1)) BEGIN
   PRINT ~Please select 1, 2, 3 or 4 and press Enter.~
   ACTION_READLN ~hpcost~
--- shadowadept/components/1_shadow_adept.tpa  2021-08-21 14:46:10.020734206 +0200
+++ shadowadept/components/1_shadow_adept.tpa   2021-08-21 14:46:22.289125022 +0200
@@ -172,11 +172,9 @@
 ///////////////////////////
 /////BETA SPELL SYSTEM/////
 ///////////////////////////
-PRINT ~Install BETA Spell Learning System?
-[1] Yes
-[2] No~
 
-OUTER_SPRINT ~spellsystem~ ~placeholder_value~
+
+OUTER_SPRINT ~spellsystem~ 2
 OUTER_WHILE (!(IS_AN_INT ~spellsystem~) OR (~spellsystem~ > 0x5) OR (~spellsystem~ < 0x1)) BEGIN
   PRINT ~Please select 1 or 2 and press Enter.~
   ACTION_READLN ~spellsystem~
