--- sod2bg2_iu/sod2bg2_iu.tp2     2021-08-20 10:51:41.531649187 +0200
+++ sod2bg2_iu/sod2bg2_iu.tp2      2021-08-20 10:53:07.822209389 +0200
@@ -4992,8 +4992,7 @@
 COPY_EXISTING ~firkra02.cre~ ~override~
   ADD_CRE_ITEM ~bdhelm01~ #0 #0 #0 ~NONE~ ~INV~		//Crown Of Lies
 	ADD_CRE_ITEM ~dtkascrl~ #0 #0 #0 ~NONE~ ~INV~		//Ancient Scroll
-		ADD_CRE_ITEM ~dtkrnd01~ #0 #0 #0 ~NONE~ ~INV~		// 50/50 Carsomyr +5 or Aster's Edge +3
-			REMOVE_CRE_ITEM ~sw2h10~	// Removing Carsomyr +5 from inventory
+		ADD_CRE_ITEM ~dtkaster~ #0 #0 #0 ~NONE~ ~INV~           // Aster's Edge +3
 				ADD_CRE_ITEM ~dtkdgbl2~ #0 #0 #0 ~NONE~ ~INV~		//Dragon's Blood
 					ADD_CRE_ITEM ~dtkrnd13~ #0 #0 #0 ~NONE~ ~INV~		//Random new spell Level 5 or lower
--- sod2bg2_iu/scripts/bg2300.baf       2021-08-22 10:40:35.197832890 +0200
+++ sod2bg2_iu/scripts/bg2300.baf       2021-08-22 10:40:43.886089154 +0200
@@ -3,7 +3,6 @@
 THEN
 RESPONSE #100
 SetGlobal("DeSpawnItem","BG2300",1)
-ActionOverride("Container 1",DestroyItem("ring08_") // Evermemory - Ring of Wizardry
 ActionOverride("Container 1",CreateItem("dtkrnd11",0,0,0)) // 66% Evermemory - 33% Kontik's Ring of Wizardry
 Continue()
 END
--- sod2bg2_iu/scripts/ar0420.baf       2021-08-22 10:44:14.264400252 +0200
+++ sod2bg2_iu/scripts/ar0420.baf       2021-08-22 10:44:21.488620087 +0200
@@ -3,7 +3,6 @@
 THEN
 RESPONSE #100
 SetGlobal("DeSpawnItem","AR0420",1)
-ActionOverride("Container 1",DestroyItem("ax1h03")) // Battle Axe of Mauletar +2
 ActionOverride("Container 1",CreateItem("dtkrnd05",0,0,0)) // Charged Battle Axe +2/Poisonous Battle Axe +2
 Continue()
 END
--- sod2bg2_iu/scripts/ar0528.baf     2021-08-22 10:45:49.295305574 +0200
+++ sod2bg2_iu/scripts/ar0528.baf       2021-08-22 10:45:57.411554980 +0200
@@ -3,7 +3,6 @@
 THEN
 RESPONSE #100
 SetGlobal("DeSpawnItem","AR0528",1)
-ActionOverride("Chest 1",DestroyItem("misc3i") // Silver Horn of Valhalla
 ActionOverride("Chest 1",CreateItem("bardic",1,0,0)) // Bardic Horn of Valhalla
 Continue()
 END
--- sod2bg2_iu/scripts/bg0102.baf     2021-08-22 10:51:10.417291032 +0200
+++ sod2bg2_iu/scripts/bg0102.baf       2021-08-22 10:51:19.229567817 +0200
@@ -4,7 +4,5 @@
 RESPONSE #100
 SetGlobal("DeSpawnItem","BG0102",1)
 ActionOverride("Container 1",CreateItem("cloaksc",0,0,0)) // Cloak of Scintillating Colors
-ActionOverride("Container 1",DestroyItem("amul12") // Lareal's Tear Necklace
-ActionOverride("Container 2",DestroyItem("clck01") // Cloak of Protection +1
 Continue()
 END
