diff --git a/Will_to_Power/components/200_thief_psion.tpa b/Will_to_Power/components/200_thief_psion.tpa
index fb93cf8..3b4bbc7 100755
--- a/Will_to_Power/components/200_thief_psion.tpa
+++ b/Will_to_Power/components/200_thief_psion.tpa
@@ -51,7 +51,7 @@ ADD_KIT ~D5_PSION~
 
 //ALIGNMENT RESTRICTIONS______________________________________________________________
 //                 LG LN LE NG TN NE CG CN CE
-~D5_PSION          1  1  1  1  1  1  0  0  0~
+~D5_PSION          1  1  1  1  1  1  1  1  1~
 
 //DUAL CLASS OPTIONS__________________________________________________________________
 //                 FT CL MA TH DR RA
@@ -91,8 +91,8 @@ ACTION_IF GAME_IS ~bgee bg2ee eet iwdee~ THEN BEGIN
 			kit_name = D5_PSION
 			backstab = ~1 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2~
 			clswpbon = ~0 0 3~
-			thiefskl = ~30 15~
-			thiefscl = ~0 100 100 100 100 100 0 0~
+			thiefskl = ~30 20~
+			thiefscl = ~100 100 100 100 100 100 0 0~
 			clsrcreq = ~1 1 1 1 1 1 1~
 	END
 END
@@ -192,7 +192,7 @@ ADD_KIT ~D5_PSION_MENU~
 
 //ALIGNMENT RESTRICTIONS______________________________________________________________
 //                 LG LN LE NG TN NE CG CN CE
-~D5_PSION_MENU     1  1  1  1  1  1  0  0  0~
+~D5_PSION_MENU     1  1  1  1  1  1  1  1  1~
 
 //DUAL CLASS OPTIONS__________________________________________________________________
 //                 FT CL MA TH DR RA
@@ -233,8 +233,8 @@ ACTION_IF GAME_IS ~bgee bg2ee eet iwdee~ THEN BEGIN
 			hpclass = ~HPWIZ~
 			backstab = ~1 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2~
 			clswpbon = ~0 0 3~
-			thiefskl = ~30 15~
-			thiefscl = ~0 100 100 100 100 100 0 0~
+			thiefskl = ~30 20~
+			thiefscl = ~100 100 100 100 100 100 0 0~
 			clsrcreq = ~1 1 1 1 1 1 1~
 	END
 END
diff --git a/Will_to_Power/components/300_fighter_psion.tpa b/Will_to_Power/components/300_fighter_psion.tpa
index 0cb0858..ea188eb 100755
--- a/Will_to_Power/components/300_fighter_psion.tpa
+++ b/Will_to_Power/components/300_fighter_psion.tpa
@@ -47,7 +47,7 @@ ADD_KIT ~D5_SOULBLADE~
 
 //ALIGNMENT RESTRICTIONS______________________________________________________________
 //                    LG LN LE NG TN NE CG CN CE
-~D5_SOULBLADE          1  1  1  1  1  1  0  0  0~
+~D5_SOULBLADE          1  1  1  1  1  1  1  1  1~
 
 //DUAL CLASS OPTIONS__________________________________________________________________
 //                 	  FT CL MA TH DR RA
@@ -183,7 +183,7 @@ ADD_KIT ~D5_SOULBLADE_MENU~
 
 //ALIGNMENT RESTRICTIONS______________________________________________________________
 //                    LG LN LE NG TN NE CG CN CE
-~D5_SOULBLADE_MENU     1  1  1  1  1  1  0  0  0~
+~D5_SOULBLADE_MENU     1  1  1  1  1  1  1  1  1~
 
 //DUAL CLASS OPTIONS__________________________________________________________________
 //                    FT CL MA TH DR RA
diff --git a/Will_to_Power/components/400_ranger_psion.tpa b/Will_to_Power/components/400_ranger_psion.tpa
index bbbde22..3c5b2f2 100755
--- a/Will_to_Power/components/400_ranger_psion.tpa
+++ b/Will_to_Power/components/400_ranger_psion.tpa
@@ -48,7 +48,7 @@ ADD_KIT ~D5_PSYPHER~
 
 //ALIGNMENT RESTRICTIONS______________________________________________________________
 //                  LG LN LE NG TN NE CG CN CE
-~D5_PSYPHER          1  1  1  1  1  1  0  0  0~
+~D5_PSYPHER          1  1  1  1  1  1  1  1  1~
 
 //DUAL CLASS OPTIONS__________________________________________________________________
 //                  FT CL MA TH DR RA
@@ -183,7 +183,7 @@ ADD_KIT ~D5_PSYPHER_MENU~
 
 //ALIGNMENT RESTRICTIONS______________________________________________________________
 //                  LG LN LE NG TN NE CG CN CE
-~D5_PSYPHER_MENU     1  1  1  1  1  1  0  0  0~
+~D5_PSYPHER_MENU     1  1  1  1  1  1  1  1  1~
 
 //DUAL CLASS OPTIONS__________________________________________________________________
 //                  FT CL MA TH DR RA
diff --git a/Will_to_Power/components/500_mage_psion.tpa b/Will_to_Power/components/500_mage_psion.tpa
index d395bfb..4471a5d 100755
--- a/Will_to_Power/components/500_mage_psion.tpa
+++ b/Will_to_Power/components/500_mage_psion.tpa
@@ -48,7 +48,7 @@ ADD_KIT ~D5_CEREBRO~
 
 //ALIGNMENT RESTRICTIONS______________________________________________________________
 //                  LG LN LE NG TN NE CG CN CE
-~D5_CEREBRO          1  1  1  1  1  1  0  0  0~
+~D5_CEREBRO          1  1  1  1  1  1  1  1  1~
 
 //DUAL CLASS OPTIONS__________________________________________________________________
 //                  FT CL MA TH DR RA
@@ -183,7 +183,7 @@ ADD_KIT ~D5_CEREBRO_MENU~
 
 //ALIGNMENT RESTRICTIONS______________________________________________________________
 //                  LG LN LE NG TN NE CG CN CE
-~D5_CEREBRO_MENU     1  1  1  1  1  1  0  0  0~
+~D5_CEREBRO_MENU     1  1  1  1  1  1  1  1  1~
 
 //DUAL CLASS OPTIONS__________________________________________________________________
 //                  FT CL MA TH DR RA
diff --git a/Will_to_Power/language/french/setup.tra b/Will_to_Power/language/french/setup.tra
index b39bddc..7bbc44d 100755
--- a/Will_to_Power/language/french/setup.tra
+++ b/Will_to_Power/language/french/setup.tra
@@ -56,9 +56,8 @@ CAPACITÉS :
 - Les psionistes commencent avec 25 points de force psionique (PFP). Ils gagnent 5 PFP supplémentaires chaque niveau jusqu'au niveau 20.
 
 Restrictions :
-- Ne dispose que de 15 points par niveau pour les compétences de voleur.
+- Ne dispose que de 20 points par niveau pour les compétences de voleur.
 - Ne peut utiliser Pickpocket ou Pose de pièges.
-- Ne peut être d'alignement chaotique.
 ~
 @301  = ~guerrier psychique~
 @302  = ~Guerrier psychique~
@@ -72,7 +71,6 @@ CAPACITÉS :
 
 Restrictions :
 - Ne peut utiliser que des armures légères.
-- Ne peut être d'alignement chaotique.
 ~
 @401  = ~psypher~
 @402  = ~Psypher~
@@ -87,7 +85,6 @@ CAPACITÉS :
 Restrictions :
 - Ne peut utiliser que des armures légères.
 - Ne peut lancer de sorts divins.
-- Ne peut être d'alignement chaotique.
 ~
 @501  = ~cérébromant~
 @502  = ~Cérébromant~
@@ -101,7 +98,6 @@ CAPACITÉS :
 
 Restrictions :
 - A les mêmes restrictions de sort et les mêmes caractéristiques pour l'apprentissage de sorts qu'un enchanteur.
-- Ne peut être d'alignement chaotique.
 ~
 //
 //@1001  = ~Télépathie~
