--- Yvette/Setup-Yvette.tp2
+++ Yvette/Setup-Yvette.tp2
@@ -535,11 +535,7 @@ Dash 1 to install original dialogues
 Dash 2 to install dialogues without action text/descriptions */
 
 
-OUTER_SPRINT ~yvettedialogues~ ~placeholder_value~
-OUTER_WHILE (!(IS_AN_INT ~yvettedialogues~) OR (~yvettedialogues~ > 0x2) OR (~yvettedialogues~ < 0x1)) BEGIN
-  PRINT @100 /* Choose 1 or 2 and confirm by "enter" */
-  ACTION_READLN ~yvettedialogues~
-END
+OUTER_SPRINT ~yvettedialogues~ 1
 
 ACTION_IF ("yvettedialogues" = 1) THEN BEGIN
   PRINT @101 /* Installing original dialogues */
