--- BG1NPCSoA/BG1NPCSoA.tp2
+++ BG1NPCSoA/BG1NPCSoA.tp2
@@ -2806,8 +2806,6 @@ END ELSE BEGIN
 END
 END
 
-AT_INTERACTIVE_EXIT ~VIEW BG1NPCSoA/readme/BG1readme.pdf~
-
 
 BEGIN @1902 DESIGNATED 2
 GROUP ~Main Mod~
@@ -5522,7 +5520,6 @@ OUTER_SPRINT ~tob_var~ ~SAFANASummonedTOB~
 EXTEND_TOP ~7XSAFA.bcs~ ~fix_tob_script.baf~ EVALUATE_BUFFER
 END
 
-AT_INTERACTIVE_EXIT ~VIEW BG1NPCSoA/readme/BG1readme.pdf~ 
 
 BEGIN @704 DESIGNATED 25
 GROUP ~Main Mod~
@@ -5725,7 +5722,6 @@ END ELSE BEGIN
 COPY ~BG1NPCSoA/portraits/Alora/vanilla-portraits~ ~override~
 END
 
-AT_INTERACTIVE_EXIT ~VIEW BG1NPCSoA/readme/BG1readme.pdf~
 
 
 
@@ -8024,7 +8020,6 @@ OUTER_SPRINT ~tob_var~ ~KagainSummonedToB~
 EXTEND_TOP ~7Xkaga.bcs~ ~fix_tob_script.baf~ EVALUATE_BUFFER
 END
 
-AT_INTERACTIVE_EXIT ~VIEW BG1NPCSoA/readme/BG1readme.pdf~
 
 
 BEGIN @1436 DESIGNATED 35
@@ -8342,7 +8337,6 @@ OUTER_SPRINT ~tob_var~ ~KagainSummonedToB~
 EXTEND_TOP ~7Xkaga.bcs~ ~fix_tob_script.baf~ EVALUATE_BUFFER
 END
 
-AT_INTERACTIVE_EXIT ~VIEW BG1NPCSoA/readme/BG1readme.pdf~
 /////////////////////////////////
 
 BEGIN @1437 DESIGNATED 40
@@ -9190,7 +9184,6 @@ OUTER_SPRINT ~tob_var~ ~MONTISummonedTOB~
 END
 
 
-AT_INTERACTIVE_EXIT ~VIEW BG1NPCSoA/readme/BG1readme.pdf~
 
 
 BEGIN @1594 DESIGNATED 50
@@ -9854,7 +9847,6 @@ OUTER_SPRINT ~tob_override~ ~7XQUAY25~
 OUTER_SPRINT ~tob_var~ ~QuayleSummonedTOB~
 END
 
-AT_INTERACTIVE_EXIT ~VIEW BG1NPCSoA/readme/BG1readme.pdf~
 
 
 // Optional component 
@@ -10996,4 +10988,4 @@ END
 
 ACTION_IF FILE_EXISTS_IN_GAME ~B7XSafa.dlg~ AND FILE_EXISTS_IN_GAME ~B7XGar.dlg~ BEGIN
 COMPILE ~%MOD_FOLDER%/crossbanter/garrick-safana.d~ USING ~%MOD_FOLDER%/tra/%LANGUAGE%/crossbanter.tra~
-END
\ No newline at end of file
+END
