--- butchery/lang/french/setup.tra
+++ butchery/lang/french/setup.tra
@@ -15,9 +15,17 @@
 
 @10 = ~Combat contre Tazok et DigDag revise par DrAzTiK~
 @20 = ~Reequilibrage de certaines creatures generiques (orques et ogres)~
-@30 = ~Reequilibrage de Tazok et de DigDag~
 
 
+@11 = ~Reequilibrage de certaines creatures generiques (orques et ogres)~
+@12 = ~Reequilibrage de Tazok et de DigDag~
+@20 = ~Groumf and Bourrinos Encounter at Firkraag's Hideout entrance~
+@21 = ~Groumpf et Bourrinos à Brynnlaw~
+@22 = ~Groumpf et Bourrinos en Outre-Terre~
+@30 = ~Revisited fight against Wraith Sarevok in the Abyss~
+@31 = ~Move Samia group into Firkraag's Entrance~
+@32 = ~Ajouter des musiques de bataille sur des zones qui n'en ont pas~
+@40 = ~Créatures de Draztik~
+@41 = ~Butchery in Irenicus Dungeon (Dwarves)~
+@42 = ~Butchery for high and low reputation traps~
+@43 = ~Butchery in the Forest of Tethyr (Animals and Druids)~
+@44 = ~Boucherie dans la sphère planaire~
+@45 = ~3 butcheries in Spellhold (Golems, Vampires and Trolls)~
+@46 = ~Butchery outside de'Arnise Keep (Trolls)~
+@47 = ~Boucherie à Lancevent~
+@48 = ~Butchery in Umar Hills (Wolves)~
+@49 = ~2 butcheries in Trademeet and outside Watcher's Keep (Animals)~
+@50 = ~2 butcheries of Golems in Suldanessellar and Watcher's Keep (Machine of Lum the Mad floor)~
+@51 = ~Butchery of Yaga-Shura revisited~
+@52 = ~Butchery in the Druid Grove area (Animals and Druids)~
+@53 = ~Boucherie au Culte de l'oeil aveugle~
+@54 = ~Butchery in Bodhi's Hideout (chapter 2, before Underdark) - Vampires~
+@55 = ~Butchery in Bodhi's Hideout (chapter 6, after Underdark) - Vampires~
+@56 = ~2 butcheries of Sahuagins in Sahuagin City and Watcher's Keep (Githyanki Encampment)~
+@57 = ~4 butcheries of spiders~
+@58 = ~6 butcheries of skeletons in lower tombs of Athkatla, Shadow Temple Land, Spellhold, Watcher's Keep, and Nyalee's Hideout (2)~
+@80 = ~Transformer BG2 en Dungeon crawler~
+@90 = ~Composant cosmétique~
 
 // WeiDU prompts
 @-1000= "Le mod %TP2_FILE_NAME% contient"
