--- gavin/gavin.tp2     2021-08-20 13:11:22.413333747 +0200
+++ gavin/gavin.tp2     2021-08-20 13:22:36.515507848 +0200
@@ -1192,11 +1192,8 @@
 REQUIRE_FILE ~override/GavinNPC.B~ @101
 
   /* Gavin's romance timer */
-PRINT @102
 
-PRINT @103
-
-OUTER_SPRINT ~gavintimer~ ~placeholder_value~
+OUTER_SPRINT ~gavintimer~ 3
 OUTER_WHILE (!(IS_AN_INT ~gavintimer~) OR (~gavintimer~ > 0x5) OR (~gavintimer~ < 0x1)) BEGIN
   PRINT @105
   ACTION_READLN ~gavintimer~
