--- SkitiaNPCs/Lib/Helga_BG2.tpa
+++ SkitiaNPCs/Lib/Helga_BG2.tpa
@@ -374,13 +374,7 @@ COMPILE	EVALUATE_BUFFER ~%MOD_FOLDER%/Dialogue/X3HelJ.d~ USING ~%MOD_FOLDER%/tra
 COMPILE	EVALUATE_BUFFER ~%MOD_FOLDER%/Dialogue/BX3Hel25.d~ USING ~%MOD_FOLDER%/tra/%LANGUAGE%/BX3Hel25.tra~
 COMPILE	EVALUATE_BUFFER ~%MOD_FOLDER%/Dialogue/X3Hel25J.d~ USING ~%MOD_FOLDER%/tra/%LANGUAGE%/X3Hel25J.tra~
 
-PRINT ~Choose a portrait for Helga (check mod folder for portraits):~
-
-PRINT ~Please choose one of the following:
-[1] Default by Acifer
-[2] Alt NWN Portrait~
-
-OUTER_SPRINT ~portrait~ ~placeholder_value~
+OUTER_SPRINT ~portrait~ 1
 OUTER_WHILE (!(IS_AN_INT ~portrait~) OR (~portrait~ > 0x2) OR (~portrait~ < 0x1)) BEGIN
   ACTION_READLN ~portrait~
 END
@@ -395,4 +389,4 @@ END
       PRINT ~Portrait: Alt NWN Portrait~
 	  END
 	  
-LOAD_TRA ~%MOD_FOLDER%/Tra/%LANGUAGE%/setup.tra~
\ No newline at end of file
+LOAD_TRA ~%MOD_FOLDER%/Tra/%LANGUAGE%/setup.tra~
--- SkitiaNPCs/Lib/Vienxay_BG2.tpa
+++ SkitiaNPCs/Lib/Vienxay_BG2.tpa
@@ -368,14 +368,8 @@ COMPILE	EVALUATE_BUFFER ~%MOD_FOLDER%/Dialogue/X3Vie25J.d~	 USING ~%MOD_FOLDER%/
  // Contains Samir and X3Drow's dialogue. 
  
    // Portraits 
-PRINT ~Choose a portrait for Vienxay (check mod folder for portraits):~
 
-PRINT ~Please choose one of the following:
-[1] Default by Crisshasart
-[2] Alt by Aeries1986
-[3] Old Version (NWN)~
-
-OUTER_SPRINT ~portrait~ ~placeholder_value~
+OUTER_SPRINT ~portrait~ 1
 OUTER_WHILE (!(IS_AN_INT ~portrait~) OR (~portrait~ > 0x3) OR (~portrait~ < 0x1)) BEGIN
   ACTION_READLN ~portrait~
 END
@@ -398,4 +392,4 @@ END
       PRINT ~Portrait: Old Version (NWN)~
 	  END
 
-LOAD_TRA ~%MOD_FOLDER%/Tra/%LANGUAGE%/setup.tra~
\ No newline at end of file
+LOAD_TRA ~%MOD_FOLDER%/Tra/%LANGUAGE%/setup.tra~
