diff --git a/dialogs/m#amber.d b/dialogs/m#amber.d
index e7f06f7..eeba47f 100644
--- amber/dialogs/m#amber.d
+++ amber/dialogs/m#amber.d
@@ -404,7 +404,6 @@ IF ~~ THEN BEGIN Kidnap2B
                  EraseJournalEntry(@51)
                  SetGlobal("M#EndMusic","GLOBAL",1)
                  SetGlobal("M#AmberKidnap","GLOBAL",9)
-                 RealSetGlobalTimer("M#KillAmberRomance","GLOBAL",21600)
                  SetGlobal("M#AmberJoined","GLOBAL",2)
                  SetGlobal("M#AmberFiveFlagons","GLOBAL",1)
                  SetGlobal("M#AmberTalk","LOCALS",12)~ EXIT
@@ -436,15 +435,13 @@ END
 
 IF ~~ THEN BEGIN 2
   SAY @5068 /* @5068 = ~Very well. I don't like this, but I'll wait here. Remember that if you don't came back soon, I probably won't be here any more.~ */
-  IF ~~ THEN DO ~RealSetGlobalTimer("M#AmberLeave","GLOBAL",7200)
-                 RealSetGlobalTimer("M#KillAmberRomance","GLOBAL",21600)
+  IF ~~ THEN DO ~RealSetGlobalTimer("M#AmberLeave","GLOBAL",720000)
                  SetGlobal("M#AmberJoined","GLOBAL",2)~ EXIT
 END
 
 IF ~~ THEN BEGIN 3
   SAY @5069 /* @5069 = ~I don't like this, but I'll find my way to the Five Flagons and wait for you there for a while.~ */
-  IF ~~ THEN DO ~RealSetGlobalTimer("M#KillAmberRomance","GLOBAL",21600)
-                 SetGlobal("M#AmberJoined","GLOBAL",2)
+  IF ~~ THEN DO ~SetGlobal("M#AmberJoined","GLOBAL",2)
                  SetGlobal("M#AmberFiveFlagons","GLOBAL",1)~ EXIT
 END
 
@@ -479,15 +476,13 @@ END
 
 IF ~~ THEN BEGIN 7
   SAY @5074 /* @5074 = ~Well, it's your loss then. I do not like this, but I'll wait here.  Remember that if you don't came back soon, I may not be here any more.~ */
-  IF ~~ THEN DO ~RealSetGlobalTimer("M#AmberLeave","GLOBAL",7200)
-                 RealSetGlobalTimer("M#KillAmberRomance","GLOBAL",21600)
+  IF ~~ THEN DO ~RealSetGlobalTimer("M#AmberLeave","GLOBAL",720000)
                  SetGlobal("M#AmberJoined","GLOBAL",2)~ EXIT
 END
 
 IF ~~ THEN BEGIN 8
   SAY @5075 /* @5075 = ~Well, it's your loss then. I do not like this, but I'll wait here.~ */
-  IF ~~ THEN DO ~RealSetGlobalTimer("M#KillAmberRomance","GLOBAL",21600)
-                 SetGlobal("M#AmberJoined","GLOBAL",2)
+  IF ~~ THEN DO ~SetGlobal("M#AmberJoined","GLOBAL",2)
                  SetGlobal("M#AmberFiveFlagons","GLOBAL",1)~ EXIT
 END
 
@@ -567,7 +562,6 @@ END
 IF ~~ THEN BEGIN 15
   SAY @5095 /* @5095 = ~After going through all the trouble of finding you again? OK, very well. But don't expect me to stay there forever.~ */
   IF ~~ THEN DO ~SetGlobal("M#EndMusic","GLOBAL",1)
-  		 RealSetGlobalTimer("M#KillAmberRomance","GLOBAL",21600)
                  SetGlobal("M#JillBracelet","GLOBAL",9)
                  SetGlobal("M#AmberQuestRemind","LOCALS",0)
                  SetGlobal("M#AmberJoined","GLOBAL",2)
diff --git a/scripts/extend/m#baldur.baf b/scripts/extend/m#baldur.baf
index b9b4871..a14043e 100644
--- amber/scripts/extend/m#baldur.baf
+++ amber/scripts/extend/m#baldur.baf
@@ -103,53 +103,6 @@ THEN
 	SetGlobal("M#AmberMatch","GLOBAL",4)
 END
 
-// Setting Amber Romance timeout variables in case Amber is kicked out of party
-IF
-	OR(2)
-	  Global("M#AmberRomanceActive","GLOBAL",1)
-	  Global("M#AmberRomanceActive","GLOBAL",2)
-	Global("M#AmberRomanceOut","GLOBAL",0)
-	OR(2)
-	  GlobalLT("M#AmberKidnap","GLOBAL",2)
-	  GlobalGT("M#AmberKidnap","GLOBAL",8)
-	!InPartyAllowDead("M#Amber")
-	!Dead("M#Amber")
-THEN
-	RESPONSE #100
-		SetGlobal("M#AmberRomanceOut","GLOBAL",1)
-		SetGlobalTimer("M#AmberOutOfParty","GLOBAL",25920)
-END
-
-IF
-	OR(2)
-	  Global("M#AmberRomanceActive","GLOBAL",1)
-	  Global("M#AmberRomanceActive","GLOBAL",2)
-	OR(2)
-	  GlobalLT("M#AmberKidnap","GLOBAL",2)
-	  GlobalGT("M#AmberKidnap","GLOBAL",8)  
-	Global("M#AmberRomanceOut","GLOBAL",1)
-	Global("M#AmberRomanceWaiting","GLOBAL",0)
-	GlobalTimerExpired("M#AmberOutOfParty","GLOBAL")
-	!InPartyAllowDead("M#Amber")
-	!Dead("M#Amber")
-THEN
-	RESPONSE #100
-		SetGlobal("M#AmberRomanceActive","GLOBAL",3)
-END
-
-IF
-	OR(2)
-	  Global("M#AmberRomanceActive","GLOBAL",1)
-	  Global("M#AmberRomanceActive","GLOBAL",2)
-	Global("M#AmberRomanceOut","GLOBAL",1)
-	InPartyAllowDead("M#Amber")
-THEN
-	RESPONSE #100
-		SetGlobal("M#AmberRomanceOut","GLOBAL",0)
-		SetGlobal("M#AmberRomanceWaiting","GLOBAL",0)
-		RealSetGlobalTimer("M#AmberRomanceTime","GLOBAL",3600)
-END
-
 // Amber returns after getting her bracelet back
 IF
 	Global("M#JillBracelet","GLOBAL",2)
