diff --git a/CtB/dialogs/translate/CB510BRD.D b/CtB/dialogs/translate/CB510BRD.D
index c65bf73..cc99d01 100644
--- CtB/dialogs/translate/CB510BRD.D
+++ CtB/dialogs/translate/CB510BRD.D
@@ -3,12 +3,12 @@ BEGIN ~CB510BRD~
 IF ~Class(LastTalkedToBy(Myself),BARD)
 Global("CbMasterBardWantsHarp","GLOBAL",0)
 Global("CbMasterBardTaught","AR0510",0)
-!GlobalGT("CbMasterBardWantsHarp","GLOBAL",0)
+/*!GlobalGT("CbMasterBardWantsHarp","GLOBAL",0)
 !GlobalGT("CbRangerDeadDear","GLOBAL",0)
 !GlobalGT("CbDruidTeachingSeeds","GLOBAL",0)
 !GlobalGT("CbTumblingWizardWantsResearch","GLOBAL",0)
 !GlobalGT("CbElvenSorceressWantsSeeds","GLOBAL",0)
-!GlobalGT("CbNorthmenWantStaff","GLOBAL",0)~ THEN BEGIN 0
+!GlobalGT("CbNorthmenWantStaff","GLOBAL",0)*/~ THEN BEGIN 0
   SAY @1
   IF ~~ THEN GOTO 1
 END
diff --git a/CtB/dialogs/translate/CB585RNG.D b/CtB/dialogs/translate/CB585RNG.D
index cd26b89..20ba8c6 100644
--- CtB/dialogs/translate/CB585RNG.D
+++ CtB/dialogs/translate/CB585RNG.D
@@ -3,12 +3,12 @@ BEGIN ~CB585RNG~
 IF ~Global("CbRangerDeadDear","GLOBAL",0)
 Class(LastTalkedToBy(Myself),RANGER)
 Global("CbRangerTaught","AR3585",0)
-!GlobalGT("CbMasterBardWantsHarp","GLOBAL",0)
+/*!GlobalGT("CbMasterBardWantsHarp","GLOBAL",0)
 !GlobalGT("CbRangerDeadDear","GLOBAL",0)
 !GlobalGT("CbDruidTeachingSeeds","GLOBAL",0)
 !GlobalGT("CbTumblingWizardWantsResearch","GLOBAL",0)
 !GlobalGT("CbElvenSorceressWantsSeeds","GLOBAL",0)
-!GlobalGT("CbNorthmenWantStaff","GLOBAL",0)~ THEN BEGIN 0
+!GlobalGT("CbNorthmenWantStaff","GLOBAL",0)*/~ THEN BEGIN 0
   SAY @1
   IF ~~ THEN GOTO 1
 END
diff --git a/CtB/dialogs/translate/CB587DR1.D b/CtB/dialogs/translate/CB587DR1.D
index adbdac0..03d8fa4 100644
--- CtB/dialogs/translate/CB587DR1.D
+++ CtB/dialogs/translate/CB587DR1.D
@@ -3,12 +3,12 @@ BEGIN ~CB587DR1~
 IF ~Global("CbDruidTeachingSeeds","GLOBAL",0)
 Class(LastTalkedToBy(Myself),DRUID)
 Gender(LastTalkedToBy(Myself),MALE)
-!GlobalGT("CbMasterBardWantsHarp","GLOBAL",0)
+/*!GlobalGT("CbMasterBardWantsHarp","GLOBAL",0)
 !GlobalGT("CbRangerDeadDear","GLOBAL",0)
 !GlobalGT("CbDruidTeachingSeeds","GLOBAL",0)
 !GlobalGT("CbTumblingWizardWantsResearch","GLOBAL",0)
 !GlobalGT("CbElvenSorceressWantsSeeds","GLOBAL",0)
-!GlobalGT("CbNorthmenWantStaff","GLOBAL",0)~ THEN BEGIN 0
+!GlobalGT("CbNorthmenWantStaff","GLOBAL",0)*/~ THEN BEGIN 0
   SAY @1
   IF ~~ THEN GOTO 1
 END
diff --git a/CtB/dialogs/translate/CBCWLTCH.D b/CtB/dialogs/translate/CBCWLTCH.D
index 3359b75..5276bce 100644
--- CtB/dialogs/translate/CBCWLTCH.D
+++ CtB/dialogs/translate/CBCWLTCH.D
@@ -6,12 +6,12 @@ Gender(LastTalkedToBy(Myself),FEMALE)
 OR(2)
 	Class(LastTalkedToBy(Myself),SORCERER)
 	Class(LastTalkedToBy(Myself),MAGE_ALL)
-!GlobalGT("CbMasterBardWantsHarp","GLOBAL",0)
+/*!GlobalGT("CbMasterBardWantsHarp","GLOBAL",0)
 !GlobalGT("CbRangerDeadDear","GLOBAL",0)
 !GlobalGT("CbDruidTeachingSeeds","GLOBAL",0)
 !GlobalGT("CbTumblingWizardWantsResearch","GLOBAL",0)
 !GlobalGT("CbElvenSorceressWantsSeeds","GLOBAL",0)
-!GlobalGT("CbNorthmenWantStaff","GLOBAL",0)~ THEN BEGIN 0
+!GlobalGT("CbNorthmenWantStaff","GLOBAL",0)*/~ THEN BEGIN 0
   SAY @1
   IF ~~ THEN REPLY @2 GOTO 7
   IF ~~ THEN REPLY @3 GOTO 4
diff --git a/CtB/dialogs/translate/CBEM25T1.D b/CtB/dialogs/translate/CBEM25T1.D
index 3663e94..165fe34 100644
--- CtB/dialogs/translate/CBEM25T1.D
+++ CtB/dialogs/translate/CBEM25T1.D
@@ -2,12 +2,12 @@ BEGIN ~CBEM25T1~
 
 IF ~Global("CbElvenSorceressWantsSeeds","GLOBAL",0)
 Class(LastTalkedToBy(Myself),MAGE_ALL)
-!GlobalGT("CbMasterBardWantsHarp","GLOBAL",0)
+/*!GlobalGT("CbMasterBardWantsHarp","GLOBAL",0)
 !GlobalGT("CbRangerDeadDear","GLOBAL",0)
 !GlobalGT("CbDruidTeachingSeeds","GLOBAL",0)
 !GlobalGT("CbTumblingWizardWantsResearch","GLOBAL",0)
 !GlobalGT("CbElvenSorceressWantsSeeds","GLOBAL",0)
-!GlobalGT("CbNorthmenWantStaff","GLOBAL",0)~ THEN BEGIN 0
+!GlobalGT("CbNorthmenWantStaff","GLOBAL",0)*/~ THEN BEGIN 0
   SAY @1
   IF ~~ THEN GOTO 1
 END
diff --git a/CtB/dialogs/translate/CBNRT020.D b/CtB/dialogs/translate/CBNRT020.D
index df947b2..ba2328a 100644
--- CtB/dialogs/translate/CBNRT020.D
+++ CtB/dialogs/translate/CBNRT020.D
@@ -5,12 +5,12 @@ IsGabber(Player1)
 Class(Player1,FIGHTER)
 Kit(Player1,0)
 Global("CbBarbariansTaught","AR3591",0)
-!GlobalGT("CbMasterBardWantsHarp","GLOBAL",0)
+/*!GlobalGT("CbMasterBardWantsHarp","GLOBAL",0)
 !GlobalGT("CbRangerDeadDear","GLOBAL",0)
 !GlobalGT("CbDruidTeachingSeeds","GLOBAL",0)
 !GlobalGT("CbTumblingWizardWantsResearch","GLOBAL",0)
 !GlobalGT("CbElvenSorceressWantsSeeds","GLOBAL",0)
-!GlobalGT("CbNorthmenWantStaff","GLOBAL",0)~ THEN BEGIN 0
+!GlobalGT("CbNorthmenWantStaff","GLOBAL",0)*/~ THEN BEGIN 0
   SAY @1
   IF ~~ THEN EXTERN ~CBNRT025~ 0
 END
