--- deArnise_romance/deArnise/Dialogue/PGNalLT.d
+++ deArnise_romance/deArnise/Dialogue/PGNalLT.d
@@ -258,8 +258,6 @@ SAY @82
 ++ @88 + t4.quitecivil
 ++ @89 + t4.solitude
 ++ @90 + t4.speakonly
-IF ~InParty("Viconia") InMyArea("Viconia") !StateCheck("Viconia",CD_STATE_NOTVALID) Global("ViconiaRomanceActive","GLOBAL",1)~ EXTERN VICONIJ t4.vic
-IF ~InParty("Jaheira") InMyArea("Jaheira") !StateCheck("Jaheira",CD_STATE_NOTVALID) Global("JaheiraRomanceActive","GLOBAL",1)~ EXTERN JAHEIRAJ t4.jae
 END
 
 IF ~~ t4.interrupt
@@ -353,14 +351,11 @@ END
 IF WEIGHT #-95 ~Global("PGNaliaLoveTalk","GLOBAL",10) Global("PGNaliaRomanceActive","GLOBAL",1)~ t5
 SAY @116 
 IF ~~ DO ~SetGlobal("PGNaliaLoveTalk","GLOBAL",11) RealSetGlobalTimer("PGNaliaRomanceTimer","GLOBAL",2700)~ + t5.always
-IF ~InParty("Jaheira") InMyArea("Jaheira") !StateCheck("Jaheira",CD_STATE_NOTVALID) Global("JaheiraRomanceActive","GLOBAL",1)~ DO ~SetGlobal("PGNaliaLoveTalk","GLOBAL",11) RealSetGlobalTimer("PGNaliaRomanceTimer","GLOBAL",2700)~ EXTERN JAHEIRAJ t5.jae
-IF ~InParty("Aerie") InMyArea("Aerie") !StateCheck("Aerie",CD_STATE_NOTVALID) Global("AerieRomanceActive","GLOBAL",1)~ DO ~SetGlobal("PGNaliaLoveTalk","GLOBAL",11) RealSetGlobalTimer("PGNaliaRomanceTimer","GLOBAL",2700)~ EXTERN AERIEJ t5.aer
 END
 
 IF ~~ t5.always
 SAY @117 
 IF ~~ + t5.always2
-IF ~InParty("Viconia") InMyArea("Viconia") !StateCheck("Viconia",CD_STATE_NOTVALID) Global("ViconiaRomanceActive","GLOBAL",1)~ EXTERN VICONIJ t5.vic
 END
 
 IF ~~ t5.always2
@@ -542,8 +537,6 @@ SAY @183
 ++ @186 DO ~SetGlobal("PGNaliaLoveTalk","GLOBAL",15) RealSetGlobalTimer("PGNaliaRomanceTimer","GLOBAL",2700)~ + t7.escape
 ++ @187 DO ~SetGlobal("PGNaliaLoveTalk","GLOBAL",15) RealSetGlobalTimer("PGNaliaRomanceTimer","GLOBAL",2700)~ + t7.harmed
 ++ @188 DO ~SetGlobal("PGNaliaLoveTalk","GLOBAL",15) RealSetGlobalTimer("PGNaliaRomanceTimer","GLOBAL",2700)~ + t7.costyou
-IF ~InParty("Viconia") InMyArea("Viconia") !StateCheck("Viconia",CD_STATE_NOTVALID) Global("ViconiaRomanceActive","GLOBAL",1)~ DO ~SetGlobal("PGNaliaLoveTalk","GLOBAL",15) RealSetGlobalTimer("PGNaliaRomanceTimer","GLOBAL",2700)~ EXTERN VICONIJ t7.vic
-IF ~InParty("Aerie") InMyArea("Aerie") !StateCheck("Aerie",CD_STATE_NOTVALID) Global("AerieRomanceActive","GLOBAL",1)~ DO ~SetGlobal("PGNaliaLoveTalk","GLOBAL",15) RealSetGlobalTimer("PGNaliaRomanceTimer","GLOBAL",2700)~ EXTERN AERIEJ t7.aer
 END
 
 IF ~~ t7.costyou
@@ -983,8 +976,6 @@ END
 IF WEIGHT #-90 ~Global("PGNaliaLoveTalk","GLOBAL",20) Global("PGNaliaRomanceActive","GLOBAL",1)~ t10
 SAY @333 
 IF ~~ DO ~SetGlobal("PGNaliaLoveTalk","GLOBAL",21) RealSetGlobalTimer("PGNaliaRomanceTimer","GLOBAL",2700)~ + t10.main
-IF ~InParty("Aerie") InMyArea("Aerie") !StateCheck("Aerie",CD_STATE_NOTVALID) Global("AerieRomanceActive","GLOBAL",1)~ DO ~SetGlobal("PGNaliaLoveTalk","GLOBAL",21) RealSetGlobalTimer("PGNaliaRomanceTimer","GLOBAL",2700)~ EXTERN AERIEJ t10.aer
-IF ~InParty("Viconia") InMyArea("Viconia") !StateCheck("Viconia",CD_STATE_NOTVALID) Global("ViconiaRomanceActive","GLOBAL",1)~ DO ~SetGlobal("PGNaliaLoveTalk","GLOBAL",21) RealSetGlobalTimer("PGNaliaRomanceTimer","GLOBAL",2700)~ EXTERN VICONIJ t10.vic
 END
 
 IF ~~ t10.main
@@ -1094,7 +1085,6 @@ IF WEIGHT #-89 ~Global("PGNaliaLoveTalk","GLOBAL",22) Global("PGNaliaRomanceActi
 SAY @374 
 = @375 
 IF ~~ DO ~SetGlobal("PGNaliaLoveTalk","GLOBAL",23) RealSetGlobalTimer("PGNaliaRomanceTimer","GLOBAL",2700)~ + t11.always
-IF ~InParty("Jaheira") InMyArea("Jaheira") !StateCheck("Jaheira",CD_STATE_NOTVALID) Global("JaheiraRomanceActive","GLOBAL",1)~ DO ~SetGlobal("PGNaliaLoveTalk","GLOBAL",23) RealSetGlobalTimer("PGNaliaRomanceTimer","GLOBAL",2700)~ EXTERN JAHEIRAJ t11.jae
 END
 
 IF ~~ t11.always
@@ -1314,7 +1304,6 @@ SAY @434
 ++ @436 DO ~SetGlobal("PGNaliaLoveTalk","GLOBAL",25) RealSetGlobalTimer("PGNaliaRomanceTimer","GLOBAL",2700)~ + t12.ignoring
 ++ @437 DO ~SetGlobal("PGNaliaLoveTalk","GLOBAL",25) RealSetGlobalTimer("PGNaliaRomanceTimer","GLOBAL",2700)~ + t12.rude
 ++ @438 DO ~SetGlobal("PGNaliaLoveTalk","GLOBAL",25) RealSetGlobalTimer("PGNaliaRomanceTimer","GLOBAL",2700)~ + t12.willanswer
-IF ~InParty("Jaheira") InMyArea("Jaheira") !StateCheck("Jaheira",CD_STATE_NOTVALID) Global("JaheiraRomanceActive","GLOBAL",1)~ DO ~SetGlobal("PGNaliaLoveTalk","GLOBAL",25) RealSetGlobalTimer("PGNaliaRomanceTimer","GLOBAL",2700)~ EXTERN JAHEIRAJ t12.jae
 END
 
 IF ~~ t12.ignoring
@@ -4129,52 +4118,7 @@ IF ~~ EXTERN NALIAJ t11.main
 IF ~Global("PGNaliaTellMeFirstLT22","LOCALS",2)~ EXIT
 
 // 12.
-
-CHAIN JAHEIRAJ t12.jae
-@1494
-== NALIAJ @1495 
-== NALIAJ @1496
-== JAHEIRAJ @1497
-== NALIAJ @1498
-== JAHEIRAJ @1499
-== NALIAJ @1500
-END
-++ @1501 EXTERN JAHEIRAJ t12.jaheira-choose
-++ @1502 EXTERN JAHEIRAJ t12.jaheira-suffocate
-++ @1503 EXTERN NALIAJ t12.nalia-clingy
-++ @1504 EXTERN NALIAJ t12.nalia-have
-
-CHAIN JAHEIRAJ t12.jaheira-choose
-@1505 
-DO ~SetGlobal("JaheiraRomanceActive","GLOBAL",3)~ 
-END
-IF ~~ EXTERN NALIAJ t12.nalia-2
-
-CHAIN JAHEIRAJ t12.jaheira-suffocate
-@1506 
-DO ~SetGlobal("JaheiraRomanceActive","GLOBAL",3)~ 
-END
-IF ~~ EXTERN NALIAJ t12.nalia-2
-
-CHAIN NALIAJ t12.nalia-clingy
-@1507
-DO ~SetGlobal("PGNaliaRomanceActive","GLOBAL",3)~ 
-EXIT
-
-CHAIN NALIAJ t12.nalia-have
-@1508 
-DO ~SetGlobal("PGNaliaRomanceActive","GLOBAL",3)~ 
-EXIT
-
-CHAIN NALIAJ t12.nalia-2
-@1509
-END
-++ @435 EXTERN NALIAJ t12.main
-++ @436 EXTERN NALIAJ t12.ignoring
-++ @437 EXTERN NALIAJ t12.rude
-++ @438 EXTERN NALIAJ t12.willanswer
-
-
+// conflict - deleted
 
 // NALIA INTERRUPTS
 
@@ -4251,18 +4195,15 @@ END
 
 CHAIN BVICONI vic4.viconia-poodle
 @1540
-DO ~SetGlobal("ViconiaRomanceActive","GLOBAL",3)~ 
 EXIT
 
 CHAIN NALIAJ vic4.nalia-back
 @1541
-DO ~SetGlobal("PGNaliaRomanceActive","GLOBAL",3)~ 
 END
 IF ~~ EXTERN BVICONI 566
 
 CHAIN NALIAJ vic4.nalia-understand
 @1542
-DO ~SetGlobal("PGNaliaRomanceActive","GLOBAL",3)~ 
 END
 IF ~~ EXTERN BVICONI 566
 
@@ -4351,69 +4292,6 @@ END
 
 // Aerie 4, final
 
-EXTEND_BOTTOM BAERIE 233
-IF ~InParty("Nalia") InMyArea("Nalia") !StateCheck("Nalia",CD_STATE_NOTVALID) Global("PGNaliaRomanceActive","GLOBAL",1)~ THEN EXTERN NALIAJ aer4
-END
-
-CHAIN NALIAJ aer4
-@1569
-== BAERIE @1570
-== NALIAJ @1571
-== BAERIE @1572
-== BAERIE @1573
-== NALIAJ @1574
-== BAERIE @1575
-== NALIAJ @1576 
-== NALIAJ @1577 
-END
-++ @1578 EXTERN NALIAJ aer4.nalia-stops
-++ @1579 EXTERN NALIAJ aer4.nalia-wants
-++ @1580 DO ~IncrementGlobal("PGNaliaPCSucks","GLOBAL",2)~ EXTERN BAERIE aer4.aerie-hysterical
-++ @1581 EXTERN BAERIE aer4.aerie-hear
-
-CHAIN NALIAJ aer4.nalia-stops
-@1582
-DO ~SetGlobal("PGNaliaRomanceActive","GLOBAL",3)~
-END
-IF ~~ EXTERN BAERIE 434
-
-CHAIN NALIAJ aer4.nalia-wants
-@1583
-DO ~SetGlobal("PGNaliaRomanceActive","GLOBAL",3)~
-END
-IF ~~ EXTERN BAERIE 434
-
-CHAIN BAERIE aer4.aerie-hysterical
-@1584
-DO ~SetGlobal("AerieRomanceActive","GLOBAL",3)
-ChangeAIScript("",DEFAULT)
-SetLeavePartyDialogFile()
-SetGlobal("KickedOut","LOCALS",1)
-LeaveParty()
-MoveGlobal("AR0607","Aerie",[1034.1034])
-Wait(2)
-RestParty()~
-EXIT
-
-CHAIN BAERIE aer4.aerie-hear
-@1585
-DO ~SetGlobal("AerieRomanceActive","GLOBAL",3)
-RestParty()~
-EXIT
-
-
-
-// LT15: interjections
-
-CHAIN JAHEIRAJ t15.jaheira
-@1586
-END
-IF ~~ EXTERN NALIAJ t15.isee
-
-CHAIN IMOEN2J t15.imoenhey
-@1587
-END
-IF ~~ EXTERN NALIAJ t15.main2
 
 // LT26: beggar
 
--- deArnise_romance/deArnise/Scripts/PGNalia.baf
+++ deArnise_romance/deArnise/Scripts/PGNalia.baf
@@ -70,51 +70,8 @@ RealSetGlobalTimer("PGNaliaRomanceTimer","GLOBAL",1200)
 SetGlobal("PGNaliaLoveTalk","GLOBAL",1)
 END
 
-// Ending the romance: concurrent
-IF 
-OR(3)
-Global("AerieRomanceActive","GLOBAL",2)
-Global("JaheiraRomanceActive","GLOBAL",2)
-Global("ViconiaRomanceActive","GLOBAL",2)
-OR(2)
-Global("PGNaliaRomanceActive","GLOBAL",1)
-Global("PGNaliaRomanceActive","GLOBAL",2)
-Global("PGNaliaAnotherRomanceWon","GLOBAL",0)
-THEN
-RESPONSE #100
-SetGlobal("PGNaliaRomanceActive","GLOBAL",3)
-SetGlobal("PGNaliaAnotherRomanceWon","GLOBAL",1)
-END
-
-IF 
-Global("PGNaliaRomanceActive","GLOBAL",2)
-Global("AerieRomanceActive","GLOBAL",1)
-Global("PGNaliaAerieRomanceLost","GLOBAL",0)
-THEN
-RESPONSE #100
-SetGlobal("AerieRomanceActive","GLOBAL",3)
-SetGlobal("PGNaliaAerieRomanceLost","GLOBAL",1)
-END
+// Ending the romance: concurrent -- deleted
 
-IF 
-Global("PGNaliaRomanceActive","GLOBAL",2)
-Global("JaheiraRomanceActive","GLOBAL",1)
-Global("PGNaliaJaheiraRomanceLost","GLOBAL",0)
-THEN
-RESPONSE #100
-SetGlobal("JaheiraRomanceActive","GLOBAL",3)
-SetGlobal("PGNaliaJaheiraRomanceLost","GLOBAL",1)
-END
-
-IF 
-Global("PGNaliaRomanceActive","GLOBAL",2)
-Global("ViconiaRomanceActive","GLOBAL",1)
-Global("PGNaliaViconiaRomanceLost","GLOBAL",0)
-THEN
-RESPONSE #100
-SetGlobal("ViconiaRomanceActive","GLOBAL",3)
-SetGlobal("PGNaliaViconiaRomanceLost","GLOBAL",1)
-END
 
 // Ending the romance: losing the keep
 IF 
