diff --git a/EET/tbl/compatibility.tbl b/EET/tbl/compatibility.tbl
index d3b215f..5fff214 100644
--- EET/tbl/compatibility.tbl
+++ EET/tbl/compatibility.tbl
@@ -26,3 +26,4 @@ SOA 1                //The Stone of Askavar (BP-BGT Worldmap version)
 KARATUR 1            //T'was a Slow Boat from Kara-Tur
 VERRSZA 0            //Verr'Sza NPC
 WHITE 0              //White NPC for BGT/Tutu/EE
+EET_TWEAKS 0    // To allow Dual XP scaling
+REACTIVITYPACK 0
+BALDURIANBARHOP 0
