diff --git a/EET/EET.tp2 b/EET/EET.tp2
index 184c704..eeb2691 100644
--- EET/EET.tp2
+++ EET/EET.tp2
@@ -286,7 +286,7 @@ END
 
 OUTER_SPRINT shell_cmd ~%lua% %MOD_FOLDER%/lib/tlk_cnt.lua~
 PRINT ~%shell_cmd%~
-AT_NOW ret_val ~%shell_cmd%~
+AT_NOW ret_val ~%shell_cmd%~ EXACT
 ACTION_IF (ret_val != 0) BEGIN
 	WARN ~Shell command failed to execute: %shell_cmd%~
 END
@@ -295,7 +295,7 @@ COPY + ~%MOD_FOLDER%/temp/tlk_cnt.txt~ ~%MOD_FOLDER%/temp~
 	SET tlk_end = tlk_end + 200000 - 1
 
 OUTER_SPRINT shell_cmd ~%lua% %MOD_FOLDER%/lib/tra_merge.lua~
-AT_NOW ret_val ~%shell_cmd%~
+AT_NOW ret_val ~%shell_cmd%~ EXACT
 ACTION_IF (ret_val != 0) BEGIN
 	WARN ~Shell command failed to execute: %shell_cmd%~
 END
