diff --git a/Emily/Emily.tp2 b/Emily/Emily.tp2
index 76662e3..38fcb47 100644
--- Emily/Emily.tp2
+++ Emily/Emily.tp2
@@ -316,3 +316,42 @@ SUBCOMPONENT @49
 REQUIRE_COMPONENT ~ArtisansKitpack.tp2~ ~0~ @50
 
 */
+BEGIN ~Emily crossmod~
+DESIGNATED 10000
+
+
+ACTION_IF FILE_EXISTS_IN_GAME ~x3KaleB.dlg~
+THEN BEGIN
+  PRINT @48 // Adding crossmod banter between Kale and Emily
+COMPILE ~Emily/dialogue/CrossMod/Kale.d~
+END 
+
+ACTION_IF FILE_EXISTS_IN_GAME ~x3VienB.dlg~
+THEN BEGIN
+  PRINT @49 // Adding crossmod banter between Emily and Vienxay
+COMPILE ~Emily/dialogue/CrossMod/Vienxay.d~
+END   
+
+ACTION_IF FILE_EXISTS_IN_GAME ~BC0AURA.dlg~
+THEN BEGIN
+  PRINT @110 // Adding crossmod banter between Aura and Emily
+COMPILE ~Emily/dialogue/CrossMod/Aura.d~
+END  
+
+ACTION_IF FILE_EXISTS_IN_GAME ~X3HelgaB.dlg~
+THEN BEGIN
+  PRINT @111 // Adding crossmod banter between Helga and Emily
+COMPILE ~Emily/dialogue/CrossMod/Helga.d~
+END  
+
+ACTION_IF FILE_EXISTS_IN_GAME ~X3RecB.dlg~
+THEN BEGIN
+  PRINT @112 // Adding crossmod banter between Recorder and Emily
+COMPILE ~Emily/dialogue/CrossMod/Recorder.d~
+END  
+
+ACTION_IF FILE_EXISTS_IN_GAME ~BC0Drake.dlg~
+THEN BEGIN
+  PRINT @113 // Adding crossmod banter between Drake and Emily
+COMPILE ~Emily/dialogue/CrossMod/Drake.d~
+END  
diff --git a/Emily/lib/emil_bg1.tpa b/Emily/lib/emil_bg1.tpa
index 3869a37..a6ad779 100644
--- Emily/lib/emil_bg1.tpa
+++ Emily/lib/emil_bg1.tpa
@@ -305,41 +305,6 @@ COPY ~%MOD_FOLDER%/creatures/X3H2.cre~ ~override/X3H2.cre~
   WRITE_ASCII 0x2cc ~X3H2~ #8   //Dialogue file     
 /////////////////////Cross Mod Content////////////////////////////
 
-ACTION_IF FILE_EXISTS_IN_GAME ~x3KaleB.dlg~
-THEN BEGIN
-  PRINT @48 // Adding crossmod banter between Kale and Emily
-COMPILE ~Emily/dialogue/CrossMod/Kale.d~
-END 
-
-ACTION_IF FILE_EXISTS_IN_GAME ~x3VienB.dlg~
-THEN BEGIN
-  PRINT @49 // Adding crossmod banter between Emily and Vienxay
-COMPILE ~Emily/dialogue/CrossMod/Vienxay.d~
-END   
-
-ACTION_IF FILE_EXISTS_IN_GAME ~BC0AURA.dlg~
-THEN BEGIN
-  PRINT @110 // Adding crossmod banter between Aura and Emily
-COMPILE ~Emily/dialogue/CrossMod/Aura.d~
-END  
-
-ACTION_IF FILE_EXISTS_IN_GAME ~X3HelgaB.dlg~
-THEN BEGIN
-  PRINT @111 // Adding crossmod banter between Helga and Emily
-COMPILE ~Emily/dialogue/CrossMod/Helga.d~
-END  
-
-ACTION_IF FILE_EXISTS_IN_GAME ~X3RecB.dlg~
-THEN BEGIN
-  PRINT @112 // Adding crossmod banter between Recorder and Emily
-COMPILE ~Emily/dialogue/CrossMod/Recorder.d~
-END  
-
-ACTION_IF FILE_EXISTS_IN_GAME ~BC0Drake.dlg~
-THEN BEGIN
-  PRINT @113 // Adding crossmod banter between Drake and Emily
-COMPILE ~Emily/dialogue/CrossMod/Drake.d~
-END  
 
 
-LOAD_TRA ~Emily/tra/English/Emily.tra~
+LOAD_TRA ~Emily/tra/English/Emily.tra~
