diff --git a/scripts/bgee.baf b/scripts/bgee.baf
index 6e5edbe..2c77d15 100644
--- evandra/scripts/bgee.baf
+++ evandra/scripts/bgee.baf
@@ -1,36 +1,5 @@
 // Romance Kills
-
-IF
-  OR(2)
-  Global("rh#EvaRomanceActive","GLOBAL",1)
-  Global("rh#EvaRomanceActive","GLOBAL",2)
-  OR(2)
-    Global("NeeraRomanceActive","GLOBAL",2)
-    Global("DornRomanceActive","GLOBAL",2)
-THEN
-  RESPONSE #100
-    SetGlobal("rh#EvaRomanceActive","GLOBAL",3)
-END
-
-IF
-  Global("rh#EvaRomanceActive","GLOBAL",2)
-  OR(2)
-    Global("NeeraRomanceActive","GLOBAL",1)
-    Global("NeeraRomanceActive","GLOBAL",2)
-THEN
-  RESPONSE #100
-    SetGlobal("NeeraRomanceActive","GLOBAL",3)
-END
-
-IF
-  Global("rh#EvaRomanceActive","GLOBAL",2)
-  OR(2)
-    Global("DornRomanceActive","GLOBAL",1)
-    Global("DornRomanceActive","GLOBAL",2)
-THEN
-  RESPONSE #100
-    SetGlobal("DornRomanceActive","GLOBAL",3)
-END
+// or not
 
 // Experience
diff --git a/scripts/rh#eva.baf b/scripts/rh#eva.baf
index b23d558..c79b402 100644
--- evandra/scripts/rh#eva.baf
+++ evandra/scripts/rh#eva.baf
@@ -135,7 +135,6 @@ END
 IF
     Global("rh#EvaRomanceCheck","GLOBAL",0)
     Gender(Player1,MALE)
-    Race(Player1,ELF)
     CheckStatGT(Player1,11,INT)
 THEN
 RESPONSE #100
@@ -707,66 +706,11 @@ END
 // ---------------------------------------------
 
 IF
-  OR(2)
-  Global("rh#EvaRomanceActive","GLOBAL",1)
-  Global("rh#EvaRomanceActive","GLOBAL",2)
-  OR(5)
-    Global("AerieRomanceActive","GLOBAL",2)
-    Global("JaheiraRomanceActive","GLOBAL",2)
-    Global("ViconiaRomanceActive","GLOBAL",2)
-    Global("NeeraRomanceActive","GLOBAL",2)
-    Global("DornRomanceActive","GLOBAL",2)
-THEN
-  RESPONSE #100
-    SetGlobal("rh#EvaRomanceActive","GLOBAL",3)
-END
-
-IF
-  Global("rh#EvaRomanceActive","GLOBAL",2)
-  OR(2)
-    Global("AerieRomanceActive","GLOBAL",1)
-    Global("AerieRomanceActive","GLOBAL",2)
-THEN
-  RESPONSE #100
-    SetGlobal("AerieRomanceActive","GLOBAL",3)
-END
-
-IF
-  Global("rh#EvaRomanceActive","GLOBAL",2)
-  OR(2)
-    Global("JaheiraRomanceActive","GLOBAL",1)
-    Global("JaheiraRomanceActive","GLOBAL",2)
-THEN
-  RESPONSE #100
-    SetGlobal("JaheiraRomanceActive","GLOBAL",3)
-END
-
-IF  
-  Global("rh#EvaRomanceActive","GLOBAL",2)
-  OR(2)
-    Global("ViconiaRomanceActive","GLOBAL",1)
-    Global("ViconiaRomanceActive","GLOBAL",2)
-THEN
-  RESPONSE #100
-    SetGlobal("ViconiaRomanceActive","GLOBAL",3)
-END
-
-IF
-  Global("rh#EvaRomanceActive","GLOBAL",2)
-  OR(2)
-    Global("NeeraRomanceActive","GLOBAL",1)
-    Global("NeeraRomanceActive","GLOBAL",2)
-THEN
-  RESPONSE #100
-    SetGlobal("NeeraRomanceActive","GLOBAL",3)
-END
-
-IF
-  Global("rh#EvaRomanceActive","GLOBAL",2)
+  Global("rh#EvaRomanceActive","GLOBAL",3)
   OR(2)
     Global("DornRomanceActive","GLOBAL",1)
     Global("DornRomanceActive","GLOBAL",2)
 THEN
   RESPONSE #100
     SetGlobal("DornRomanceActive","GLOBAL",3)
