diff --git Fade/Dialogues/BE3FADE.d Fade/Dialogues/BE3FADE.d
index 62aaaa1..198ab07 100644
--- Fade/Dialogues/BE3FADE.d
+++ Fade/Dialogues/BE3FADE.d
@@ -5488,7 +5488,7 @@ EXIT
 CHAIN
 IF ~~ THEN BE3FADE fadevsviconia3_interest
 @3956 DO ~SetGlobal("E3FADEROMANCEACTIVE","GLOBAL",3)~
-== BVICONI @3957 DO ~SetGlobal("VICONIAROMANCEACTIVE","GLOBAL",3)~
+== BVICONI @3957 DO ~~
 EXIT
 
 CHAIN
@@ -5505,7 +5505,7 @@ EXIT
 
 CHAIN
 IF ~~ THEN BJAHEIR fadevsjaheira_ask
-@3965 DO ~SetGlobal("JAHEIRAROMANCEACTIVE","GLOBAL",3)~ 
+@3965 DO ~~ 
 == BE3FADE @3966
 END
 IF ~~ THEN REPLY @2359 EXTERN BE3FADE lt14_recently
@@ -5940,7 +5940,7 @@ EXIT
 APPEND BAERIE
 IF ~~ THEN BEGIN fadevsaerie_like
 SAY @4222
-IF ~~ THEN DO ~SetGlobal("AERIEROMANCEACTIVE","GLOBAL",3)~ EXTERN BE3FADE fadevsaerie_like_2
+IF ~~ THEN DO ~~ EXTERN BE3FADE fadevsaerie_like_2
 END
 END
 
@@ -5981,38 +5981,38 @@ END
 
 IF ~~ THEN BEGIN fadevsviconia3_saying
 SAY @3957
-IF ~~ THEN DO ~SetGlobal("VICONIAROMANCEACTIVE","GLOBAL",3)~ EXTERN BE3FADE fadevsviconia3_saying_2
+IF ~~ THEN DO ~~ EXTERN BE3FADE fadevsviconia3_saying_2
 END
 END
 
 INTERJECT BE3FADE lt4_start e3fadevsaerie
 == BAERIE IF ~InParty("Aerie") InMyArea("Aerie")
 !StateCheck("Aerie",CD_STATE_NOTVALID)
-Global("AERIEROMANCEACTIVE","GLOBAL",1)~ THEN @4229
+False()~ THEN @4229
 == BE3FADE IF ~InParty("Aerie") InMyArea("Aerie")
 !StateCheck("Aerie",CD_STATE_NOTVALID)
-Global("AERIEROMANCEACTIVE","GLOBAL",1)~ THEN @4205
+False()~ THEN @4205
 == BAERIE IF ~InParty("Aerie") InMyArea("Aerie")
 !StateCheck("Aerie",CD_STATE_NOTVALID)
-Global("AERIEROMANCEACTIVE","GLOBAL",1)~ THEN @4230
+False()~ THEN @4230
 == BE3FADE IF ~InParty("Aerie") InMyArea("Aerie")
 !StateCheck("Aerie",CD_STATE_NOTVALID)
-Global("AERIEROMANCEACTIVE","GLOBAL",1)~ THEN @4231
+False()~ THEN @4231
 == BAERIE IF ~InParty("Aerie") InMyArea("Aerie")
 !StateCheck("Aerie",CD_STATE_NOTVALID)
-Global("AERIEROMANCEACTIVE","GLOBAL",1)~ THEN @4232
+False()~ THEN @4232
 == BE3FADE IF ~InParty("Aerie") InMyArea("Aerie")
 !StateCheck("Aerie",CD_STATE_NOTVALID)
-Global("AERIEROMANCEACTIVE","GLOBAL",1)~ THEN @4233
+False()~ THEN @4233
 == BAERIE IF ~InParty("Aerie") InMyArea("Aerie")
 !StateCheck("Aerie",CD_STATE_NOTVALID)
-Global("AERIEROMANCEACTIVE","GLOBAL",1)~ THEN @4234
+False()~ THEN @4234
 == BE3FADE IF ~InParty("Aerie") InMyArea("Aerie")
 !StateCheck("Aerie",CD_STATE_NOTVALID)
-Global("AERIEROMANCEACTIVE","GLOBAL",1)~ THEN @4235
+False()~ THEN @4235
 == BAERIE IF ~InParty("Aerie") InMyArea("Aerie")
 !StateCheck("Aerie",CD_STATE_NOTVALID)
-Global("AERIEROMANCEACTIVE","GLOBAL",1)~ THEN @4236
+False()~ THEN @4236
 END
 IF ~~ THEN REPLY @4237 GOTO fadevsaerie_me
 IF ~~ THEN REPLY @4238 EXTERN BAERIE fadevsaerie_like
@@ -6020,19 +6020,19 @@ IF ~~ THEN REPLY @4238 EXTERN BAERIE fadevsaerie_like
 INTERJECT BE3FADE lt10_start e3fadevsviconia1
 == BVICONI IF ~InParty("Viconia") InMyArea("Viconia")
 !StateCheck("Viconia",CD_STATE_NOTVALID)
-Global("VICONIAROMANCEACTIVE","GLOBAL",1)~ THEN @4239
+False()~ THEN @4239
 == BE3FADE IF ~InParty("Viconia") InMyArea("Viconia")
 !StateCheck("Viconia",CD_STATE_NOTVALID)
-Global("VICONIAROMANCEACTIVE","GLOBAL",1)~ THEN @4240
+False()~ THEN @4240
 == BVICONI IF ~InParty("Viconia") InMyArea("Viconia")
 !StateCheck("Viconia",CD_STATE_NOTVALID)
-Global("VICONIAROMANCEACTIVE","GLOBAL",1)~ THEN @4241
+False()~ THEN @4241
 == BE3FADE IF ~InParty("Viconia") InMyArea("Viconia")
 !StateCheck("Viconia",CD_STATE_NOTVALID)
-Global("VICONIAROMANCEACTIVE","GLOBAL",1)~ THEN @4242
+False()~ THEN @4242
 == BVICONI IF ~InParty("Viconia") InMyArea("Viconia")
 !StateCheck("Viconia",CD_STATE_NOTVALID)
-Global("VICONIAROMANCEACTIVE","GLOBAL",1)~ THEN @4243
+False()~ THEN @4243
 END
 IF ~~ THEN REPLY @4244 GOTO lt10_childhood
 IF ~~ THEN REPLY @4245 GOTO lt10_waned
@@ -6040,25 +6040,25 @@ IF ~~ THEN REPLY @4245 GOTO lt10_waned
 INTERJECT BE3FADE lt20_start e3fadevsviconia3
 == BVICONI IF ~InParty("Viconia") InMyArea("Viconia")
 !StateCheck("Viconia",CD_STATE_NOTVALID)
-Global("VICONIAROMANCEACTIVE","GLOBAL",1)~ THEN @4246
+False()~ THEN @4246
 == BE3FADE IF ~InParty("Viconia") InMyArea("Viconia")
 !StateCheck("Viconia",CD_STATE_NOTVALID)
-Global("VICONIAROMANCEACTIVE","GLOBAL",1)~ THEN @4247
+False()~ THEN @4247
 == BVICONI IF ~InParty("Viconia") InMyArea("Viconia")
 !StateCheck("Viconia",CD_STATE_NOTVALID)
-Global("VICONIAROMANCEACTIVE","GLOBAL",1)~ THEN @4248
+False()~ THEN @4248
 == BE3FADE IF ~InParty("Viconia") InMyArea("Viconia")
 !StateCheck("Viconia",CD_STATE_NOTVALID)
-Global("VICONIAROMANCEACTIVE","GLOBAL",1)~ THEN @4249
+False()~ THEN @4249
 == BVICONI IF ~InParty("Viconia") InMyArea("Viconia")
 !StateCheck("Viconia",CD_STATE_NOTVALID)
-Global("VICONIAROMANCEACTIVE","GLOBAL",1)~ THEN @4250
+False()~ THEN @4250
 == BE3FADE IF ~InParty("Viconia") InMyArea("Viconia")
 !StateCheck("Viconia",CD_STATE_NOTVALID)
-Global("VICONIAROMANCEACTIVE","GLOBAL",1)~ THEN @4251
+False()~ THEN @4251
 == BVICONI IF ~InParty("Viconia") InMyArea("Viconia")
 !StateCheck("Viconia",CD_STATE_NOTVALID)
-Global("VICONIAROMANCEACTIVE","GLOBAL",1)~ THEN @4252
+False()~ THEN @4252
 END
 IF ~~ THEN REPLY @4253 GOTO fadevsviconia3_point
 IF ~~ THEN REPLY @4254 EXTERN BVICONI fadevsviconia3_saying
@@ -6067,22 +6067,22 @@ IF ~~ THEN REPLY @4255 GOTO fadevsviconia3_interest
 INTERJECT BE3FADE lt14_start e3fadevsjaheira
 == BJAHEIR IF ~InParty("Jaheira") InMyArea("Jaheira")
 !StateCheck("Jaheira",CD_STATE_NOTVALID)
-Global("JAHEIRAROMANCEACTIVE","GLOBAL",1)~ THEN @4256
+False()~ THEN @4256
 == BE3FADE IF ~InParty("Jaheira") InMyArea("Jaheira")
 !StateCheck("Jaheira",CD_STATE_NOTVALID)
-Global("JAHEIRAROMANCEACTIVE","GLOBAL",1)~ THEN @4257
+False()~ THEN @4257
 == BJAHEIR IF ~InParty("Jaheira") InMyArea("Jaheira")
 !StateCheck("Jaheira",CD_STATE_NOTVALID)
-Global("JAHEIRAROMANCEACTIVE","GLOBAL",1)~ THEN @4258
+False()~ THEN @4258
 == BE3FADE IF ~InParty("Jaheira") InMyArea("Jaheira")
 !StateCheck("Jaheira",CD_STATE_NOTVALID)
-Global("JAHEIRAROMANCEACTIVE","GLOBAL",1)~ THEN @4259
+False()~ THEN @4259
 == BJAHEIR IF ~InParty("Jaheira") InMyArea("Jaheira")
 !StateCheck("Jaheira",CD_STATE_NOTVALID)
-Global("JAHEIRAROMANCEACTIVE","GLOBAL",1)~ THEN @4260
+False()~ THEN @4260
 == BE3FADE IF ~InParty("Jaheira") InMyArea("Jaheira")
 !StateCheck("Jaheira",CD_STATE_NOTVALID)
-Global("JAHEIRAROMANCEACTIVE","GLOBAL",1)~ THEN @4261
+False()~ THEN @4261
 END
 IF ~~ THEN REPLY @4262 GOTO fadevsjaheira_apologise
 IF ~~ THEN REPLY @4263 EXTERN BJAHEIR fadevsjaheira_slack
diff --git Fade/Dialogues/EE.d Fade/Dialogues/EE.d
index f6c3c9d..f0b1303 100644
--- Fade/Dialogues/EE.d
+++ Fade/Dialogues/EE.d
@@ -157,7 +157,7 @@ END
 IF ~~ THEN BEGIN fadeneera_pickfade
 SAY @6763
 = @6764
-IF ~~ THEN DO ~SetGlobal("NEERAROMANCEACTIVE","GLOBAL",3) SetGlobal("E3NeeraFadeCon","GLOBAL",3)~ EXIT
+IF ~~ THEN DO ~SetGlobal("E3NeeraFadeCon","GLOBAL",3)~ EXIT
 END
 END
 
@@ -269,7 +269,7 @@ END
 
 IF ~~ THEN BEGIN fadehexxat_teamfade
 SAY @6812
-IF ~~ THEN DO ~SetGlobal("HEXXATROMANCEACTIVE","GLOBAL",3) SetGlobal("E3FadeHexxatCon","GLOBAL",3)~EXIT
+IF ~~ THEN DO ~SetGlobal("E3FadeHexxatCon","GLOBAL",3)~EXIT
 END
 END
 
@@ -417,8 +417,8 @@ END
 IF ~~ THEN BEGIN faderasaad_notwhatyouthink
 SAY @6882
 = @6883
-IF ~~ THEN REPLY @6884 DO ~SetGlobal("RASAADROMANCEACTIVE","GLOBAL",3) SetGlobal("E3FadeRASAADCon","GLOBAL",3)~ EXIT
-IF ~~ THEN REPLY @6885 DO ~SetGlobal("RASAADROMANCEACTIVE","GLOBAL",3) SetGlobal("E3FadeRASAADCon","GLOBAL",3)~ EXIT
+IF ~~ THEN REPLY @6884 DO ~SetGlobal("E3FadeRASAADCon","GLOBAL",3)~ EXIT
+IF ~~ THEN REPLY @6885 DO ~SetGlobal("E3FadeRASAADCon","GLOBAL",3)~ EXIT
 END
 END
 
@@ -585,7 +585,7 @@ END
 
 IF ~~ THEN BEGIN fadedorn_compromise
 SAY @6962
-IF ~~ THEN DO ~SetGlobal("DORNROMANCEACTIVE","GLOBAL",3) SetGlobal("E3FadeDORNCon","GLOBAL",3)~ EXIT
+IF ~~ THEN DO ~SetGlobal("E3FadeDORNCon","GLOBAL",3)~ EXIT
 END
 
 IF ~~ THEN BEGIN fadedorn_dontask
@@ -641,4 +641,4 @@ DO ~SetGlobal("E3FADEDORNTALKTOB","GLOBAL",2)~
   ==BE3FAD25 @6984
   ==BDORN25 @6985
   ==BE3FAD25 @6986
-  EXIT
\ No newline at end of file
+  EXIT
diff --git Fade/Dialogues/FADE.d Fade/Dialogues/FADE.d
index 77a81ca..1e94a35 100644
--- Fade/Dialogues/FADE.d
+++ Fade/Dialogues/FADE.d
@@ -360,7 +360,7 @@ END
 
 IF ~~ THEN BEGIN kidnap_2_sorry
 SAY @83
-IF ~~ THEN DO ~SetGlobal("E3FADEROMANCEACTIVE","GLOBAL",3)
+IF ~~ THEN DO ~
 SetGlobal("E3FADEJOINED","LOCALS",0)
 MoveGlobal("AR0307","E3Fade",[4836.2136])~ EXIT
 END
diff --git Fade/Dialogues/RE.d Fade/Dialogues/RE.d
index a4585ba..7615059 100644
--- Fade/Dialogues/RE.d
+++ Fade/Dialogues/RE.d
@@ -21,7 +21,7 @@ END
 
 IF ~~ THEN BEGIN fade_TALMISS_greatsex
 SAY @6992
-IF~~THEN DO ~SetGlobal("E3FADEROMANCEACTIVE","GLOBAL",3) SetGlobal("E3FadeREAda","GLOBAL",2)~ EXIT
+IF~~THEN DO ~SetGlobal("E3FadeREAda","GLOBAL",2)~ EXIT
 END
 
 IF ~~ THEN BEGIN fade_TALMISS_getout
@@ -249,4 +249,4 @@ END
 
 
 
-END
\ No newline at end of file
+END
diff --git Fade/Scripts/E3Aerie.baf Fade/Scripts/E3Aerie.baf
index 7dd9969..0276cd6 100644
--- Fade/Scripts/E3Aerie.baf
+++ Fade/Scripts/E3Aerie.baf
@@ -1,6 +1,6 @@
 IF
   InParty(Myself)
-  Global("AERIEROMANCEACTIVE","GLOBAL",2)
+  False()
   OR(2)
        Global("E3FADEROMANCEACTIVE","GLOBAL",1)
        Global("E3FADEROMANCEACTIVE","GLOBAL",2)
diff --git Fade/Scripts/E3Fade.baf Fade/Scripts/E3Fade.baf
index 7cc91fb..b94fbdc 100644
--- Fade/Scripts/E3Fade.baf
+++ Fade/Scripts/E3Fade.baf
@@ -21,33 +21,30 @@ IF
   InParty(Myself)
   Global("E3FADEROMANCEACTIVE","GLOBAL",2)
   OR(2)
-       Global("AERIEROMANCEACTIVE","GLOBAL",1)
-       Global("AERIEROMANCEACTIVE","GLOBAL",2)
+       False()
+       False()
 THEN
     RESPONSE #100
-       SetGlobal("AERIEROMANCEACTIVE","GLOBAL",3)
 END
 
 IF
   InParty(Myself)
   Global("E3FADEROMANCEACTIVE","GLOBAL",2)
   OR(2)
-       Global("JAHEIRAROMANCEACTIVE","GLOBAL",1)
-       Global("JAHEIRAROMANCEACTIVE","GLOBAL",2)
+       False()
+       False()
 THEN
     RESPONSE #100
-        SetGlobal("JAHEIRAROMANCEACTIVE","GLOBAL",3)
 END
 
 IF
   InParty(Myself)
   Global("E3FADEROMANCEACTIVE","GLOBAL",2)
   OR(2)
-       Global("VICONIAROMANCEACTIVE","GLOBAL",1)
-       Global("VICONIAROMANCEACTIVE","GLOBAL",2)
+       False()
+       False()
 THEN
     RESPONSE #100
-        SetGlobal("VICONIAROMANCEACTIVE","GLOBAL",3)
 END
 
 IF
diff --git Fade/Scripts/E3Jaheira.baf Fade/Scripts/E3Jaheira.baf
index d7cdb4c..0276cd6 100644
--- Fade/Scripts/E3Jaheira.baf
+++ Fade/Scripts/E3Jaheira.baf
@@ -1,6 +1,6 @@
 IF
   InParty(Myself)
-  Global("JAHEIRAROMANCEACTIVE","GLOBAL",2)
+  False()
   OR(2)
        Global("E3FADEROMANCEACTIVE","GLOBAL",1)
        Global("E3FADEROMANCEACTIVE","GLOBAL",2)
diff --git Fade/Scripts/E3Viconia.baf Fade/Scripts/E3Viconia.baf
index ef78d62..fe0d375 100644
--- Fade/Scripts/E3Viconia.baf
+++ Fade/Scripts/E3Viconia.baf
@@ -1,6 +1,6 @@
 IF
   InParty(Myself)
-  Global("VICONIAROMANCEACTIVE","GLOBAL",2)
+  False()
   OR(2)
        Global("E3FADEROMANCEACTIVE","GLOBAL",1)
        Global("E3FADEROMANCEACTIVE","GLOBAL",2)
@@ -16,7 +16,7 @@ IF
   GlobalGT("LOVETALK","LOCALS",6)
   GlobalGT("E3LOVETALK","GLOBAL",10)
   GlobalLT("E3LOVETALK","GLOBAL",20)
-  Global("VICONIAROMANCEACTIVE","GLOBAL",1)
+  False()
   Global("E3FADEROMANCEACTIVE","GLOBAL",1)
   Global("E3FADEVSVICONIASTART","GLOBAL",0)
 THEN
@@ -36,7 +36,7 @@ IF
   Global("E3FADEVSVICONIA2","GLOBAL",0)
   GlobalGT("E3LOVETALK","GLOBAL",10)
   GlobalLT("E3LOVETALK","GLOBAL",20)
-  Global("VICONIAROMANCEACTIVE","GLOBAL",1)
+  False()
   Global("E3FADEROMANCEACTIVE","GLOBAL",1)
 THEN
     RESPONSE #100
diff --git Fade/Scripts/EE.baf Fade/Scripts/EE.baf
index be8a47d..73669cf 100644
--- Fade/Scripts/EE.baf
+++ Fade/Scripts/EE.baf
@@ -12,8 +12,8 @@ Global("E3FadeNeeraCon","GLOBAL",0)
 GlobalGT("NeeraLovetalks","GLOBAL",10)
 GlobalGT("E3LOVETALK","GLOBAL",8)
 OR(2)
-Global("NeeraRomanceActive","GLOBAL",1)
-Global("NeeraRomanceActive","GLOBAL",2)
+False()
+False()
 OR(2)
 Global("E3FADEROMANCEACTIVE","GLOBAL",1)
 Global("E3FADEROMANCEACTIVE","GLOBAL",2)
@@ -34,8 +34,8 @@ RealGlobalTimerExpired("E3FadeNeeraConTime","GLOBAL")
 GlobalGT("NeeraLovetalks","GLOBAL",10)
 GlobalGT("E3LOVETALK","GLOBAL",8)
 OR(2)
-Global("NeeraRomanceActive","GLOBAL",1)
-Global("NeeraRomanceActive","GLOBAL",2)
+False()
+False()
 OR(2)
 Global("E3FADEROMANCEACTIVE","GLOBAL",1)
 Global("E3FADEROMANCEACTIVE","GLOBAL",2)
@@ -62,8 +62,8 @@ Global("E3NeeraFadeCon","GLOBAL",0)
 GlobalGT("NeeraLovetalks","GLOBAL",10)
 GlobalGT("E3LOVETALK","GLOBAL",12)
 OR(2)
-Global("NeeraRomanceActive","GLOBAL",1)
-Global("NeeraRomanceActive","GLOBAL",2)
+False()
+False()
 OR(2)
 Global("E3FADEROMANCEACTIVE","GLOBAL",1)
 Global("E3FADEROMANCEACTIVE","GLOBAL",2)
@@ -84,8 +84,8 @@ RealGlobalTimerExpired("E3NeeraFadeConTime","GLOBAL")
 GlobalGT("NeeraLovetalks","GLOBAL",10)
 GlobalGT("E3LOVETALK","GLOBAL",12)
 OR(2)
-Global("NeeraRomanceActive","GLOBAL",1)
-Global("NeeraRomanceActive","GLOBAL",2)
+False()
+False()
 OR(2)
 Global("E3FADEROMANCEACTIVE","GLOBAL",1)
 Global("E3FADEROMANCEACTIVE","GLOBAL",2)
@@ -114,8 +114,8 @@ Global("E3FadeHexxatCon","GLOBAL",0)
 GlobalGT("HEXXATLovetalks","GLOBAL",10)
 GlobalGT("E3LOVETALK","GLOBAL",8)
 OR(2)
-Global("HexxatRomanceActive","GLOBAL",1)
-Global("HexxatRomanceActive","GLOBAL",2)
+False()
+False()
 OR(2)
 Global("E3FADEROMANCEACTIVE","GLOBAL",1)
 Global("E3FADEROMANCEACTIVE","GLOBAL",2)
@@ -135,8 +135,8 @@ Global("E3FadeHexxatCon","GLOBAL",1)
 GlobalGT("HEXXATLovetalks","GLOBAL",10)
 GlobalGT("E3LOVETALK","GLOBAL",8)
 OR(2)
-Global("HexxatRomanceActive","GLOBAL",1)
-Global("HexxatRomanceActive","GLOBAL",2)
+False()
+False()
 OR(2)
 Global("E3FADEROMANCEACTIVE","GLOBAL",1)
 Global("E3FADEROMANCEACTIVE","GLOBAL",2)
@@ -164,8 +164,8 @@ Global("E3FadeRASAADCon","GLOBAL",0)
 GlobalGT("RASAADLovetalks","GLOBAL",10)
 GlobalGT("E3LOVETALK","GLOBAL",8)
 OR(2)
-Global("RASAADRomanceActive","GLOBAL",1)
-Global("RASAADRomanceActive","GLOBAL",2)
+False()
+False()
 OR(2)
 Global("E3FADEROMANCEACTIVE","GLOBAL",1)
 Global("E3FADEROMANCEACTIVE","GLOBAL",2)
@@ -185,8 +185,8 @@ Global("E3FadeRASAADCon","GLOBAL",1)
 GlobalGT("RASAADLovetalks","GLOBAL",10)
 GlobalGT("E3LOVETALK","GLOBAL",8)
 OR(2)
-Global("RASAADRomanceActive","GLOBAL",1)
-Global("RASAADRomanceActive","GLOBAL",2)
+False()
+False()
 OR(2)
 Global("E3FADEROMANCEACTIVE","GLOBAL",1)
 Global("E3FADEROMANCEACTIVE","GLOBAL",2)
@@ -214,8 +214,8 @@ Global("E3FadeDORNCon","GLOBAL",0)
 GlobalGT("DORNLovetalks","GLOBAL",10)
 GlobalGT("E3LOVETALK","GLOBAL",8)
 OR(2)
-Global("DORNRomanceActive","GLOBAL",1)
-Global("DORNRomanceActive","GLOBAL",2)
+False()
+False()
 OR(2)
 Global("E3FADEROMANCEACTIVE","GLOBAL",1)
 Global("E3FADEROMANCEACTIVE","GLOBAL",2)
@@ -235,8 +235,8 @@ Global("E3FadeDORNCon","GLOBAL",1)
 GlobalGT("DORNLovetalks","GLOBAL",10)
 GlobalGT("E3LOVETALK","GLOBAL",8)
 OR(2)
-Global("DORNRomanceActive","GLOBAL",1)
-Global("DORNRomanceActive","GLOBAL",2)
+False()
+False()
 OR(2)
 Global("E3FADEROMANCEACTIVE","GLOBAL",1)
 Global("E3FADEROMANCEACTIVE","GLOBAL",2)
@@ -250,4 +250,4 @@ Global("E3FadeDORNCon","GLOBAL",2)
 THEN
 RESPONSE #100
 StartDialogue("E3FADEJ",Player1)
-END
\ No newline at end of file
+END
