diff --git a/Helga/setup-Helga.TP2 b/Helga/setup-Helga.TP2
index 8911701..43f0f03 100644
--- Helga/setup-Helga.TP2
+++ Helga/setup-Helga.TP2
@@ -298,35 +298,6 @@ END
 
 ////////////////////////////////////////////////
 
-ACTION_IF FILE_EXISTS_IN_GAME ~x3milyB.dlg~
-THEN BEGIN
-  PRINT @56
-COMPILE ~Helga/Dialogue/Crossmod/Emily.d~
-END 
-
-ACTION_IF FILE_EXISTS_IN_GAME ~x3KaleB.dlg~
-THEN BEGIN
-  PRINT @57
-COMPILE ~Helga/Dialogue/Crossmod/Kale.d~
-END 
-
-ACTION_IF FILE_EXISTS_IN_GAME ~x3VienB.dlg~
-THEN BEGIN
-  PRINT @58
-COMPILE ~Helga/Dialogue/Crossmod/Vienxay.d~
-END 
-
-ACTION_IF FILE_EXISTS_IN_GAME ~x3RecB.dlg~
-THEN BEGIN
-  PRINT @59
-COMPILE ~Helga/Dialogue/Crossmod/Recorder.d~
-END 
-
-ACTION_IF FILE_EXISTS_IN_GAME ~BC0Aura.dlg~
-THEN BEGIN
-  PRINT @60
-COMPILE ~Helga/Dialogue/Crossmod/Aura.d~
-END 
 
 PRINT @61
@@ -500,3 +471,36 @@ COPY ~%MOD_FOLDER%/Cre/X3HFC1.cre~ ~override/X3Helga.cre~
   SAY PICKED_POCKET ~~ []	  
   WRITE_SHORT 0x246 (IDS_OF_SYMBOL (~kit~ ~X3HFC~))
   
+
+BEGIN ~Helga Crossmod~
+DESIGNATED 10000
+
+ACTION_IF FILE_EXISTS_IN_GAME ~x3milyB.dlg~
+THEN BEGIN
+  PRINT @56
+COMPILE ~Helga/Dialogue/Crossmod/Emily.d~
+END 
+
+ACTION_IF FILE_EXISTS_IN_GAME ~x3KaleB.dlg~
+THEN BEGIN
+  PRINT @57
+COMPILE ~Helga/Dialogue/Crossmod/Kale.d~
+END 
+
+ACTION_IF FILE_EXISTS_IN_GAME ~x3VienB.dlg~
+THEN BEGIN
+  PRINT @58
+COMPILE ~Helga/Dialogue/Crossmod/Vienxay.d~
+END 
+
+ACTION_IF FILE_EXISTS_IN_GAME ~x3RecB.dlg~
+THEN BEGIN
+  PRINT @59
+COMPILE ~Helga/Dialogue/Crossmod/Recorder.d~
+END 
+
+ACTION_IF FILE_EXISTS_IN_GAME ~BC0Aura.dlg~
+THEN BEGIN
+  PRINT @60
+COMPILE ~Helga/Dialogue/Crossmod/Aura.d~
+END 
