--- Helga/setup-Helga.TP2
+++ Helga/setup-Helga.TP2
@@ -332,10 +332,7 @@ PRINT @61
 
 PRINT @62
 
-OUTER_SPRINT ~portrait~ @63
-OUTER_WHILE (!(IS_AN_INT ~portrait~) OR (~portrait~ > 0x2) OR (~portrait~ < 0x1)) BEGIN
-  ACTION_READLN ~portrait~
-END
+OUTER_SPRINT ~portrait~ 1
       ACTION_IF ("portrait" = 1) THEN BEGIN
                COPY ~%MOD_FOLDER%/Portraits/HelgaL.bmp~ ~override/HelgaL.bmp~
                COPY ~%MOD_FOLDER%/Portraits/HelgaM.bmp~ ~override/HelgaM.bmp~
