diff --git a/ntotsc/language/french/worldmap.tra b/ntotsc/language/french/worldmap.tra
index f016076..cb0231a 100644
--- ntotsc/language/french/worldmap.tra
+++ ntotsc/language/french/worldmap.tra
@@ -5,6 +5,7 @@
 @NT_783004  = "Bois###des###morts"
 @NT_783005  = "Citadelle###du###Nord"
 @NT_783006  = "Temple###de###la###Main###Noire"
+@NT_783007  = "Île###de###la###Sirène"
 
 #BGEE: 22853
 @NT_783010  = "Barbe###d'Ulgoth"
