diff --git a/vladsime/main/scripts/ttsime.baf b/vladsime/main/scripts/ttsime.baf
index 9d058bb..49e7225 100644
--- TS_Sime/vladsime/main/scripts/ttsime.baf
+++ TS_Sime/vladsime/main/scripts/ttsime.baf
@@ -126,193 +126,23 @@ THEN
 END
 
 // Romance Autokills
-
-IF
-	Global("ttSimeRomanceActive","GLOBAL",1)
-	OR(4)
-		Global("AnomenRomanceActive","GLOBAL",2)
-		Global("RasaadRomanceActive","GLOBAL",2)
-		Global("NeeraRomanceActive","GLOBAL",2)
-		Global("HexxatRomanceActive","GLOBAL",2)
-THEN
-	RESPONSE #100
-		SetGlobal("ttSimeRomanceActive","GLOBAL",3)
-END
-
-IF
-	Global("ttSimeRomanceActive","GLOBAL",2)
-	Global("AnomenRomanceActive","GLOBAL",1)
-THEN
-	RESPONSE #100
-		SetGlobal("AnomenRomanceActive","GLOBAL",3)
-END
-
-IF
-	Global("ttSimeRomanceActive","GLOBAL",2)
-	Global("RasaadRomanceActive","GLOBAL",1)
-THEN
-	RESPONSE #100
-		SetGlobal("RasaadRomanceActive","GLOBAL",3)
-END
-
-IF
-	Global("ttSimeRomanceActive","GLOBAL",2)
-	Global("NeeraRomanceActive","GLOBAL",1)
-THEN
-	RESPONSE #100
-		SetGlobal("NeeraRomanceActive","GLOBAL",3)
-END
-
-IF
-	Global("ttSimeRomanceActive","GLOBAL",2)
-	Global("HexxatRomanceActive","GLOBAL",1)
-THEN
-	RESPONSE #100
-		SetGlobal("HexxatRomanceActive","GLOBAL",3)
-END
+// deleted
 
 ///////////////////////////
 // Aerie Romance Conflict
 ///////////////////////////
 
-IF
-	InParty(Myself)
-	InParty("Aerie")
-	See(Player1)
-	See("Aerie")
-	!StateCheck("Aerie",CD_STATE_NOTVALID)
-	!See([ENEMY])
-	CombatCounter(0)
-	Global("ttSimeRomanceActive","GLOBAL",2)
-	OR(2)
-		Global("AerieRomanceActive","GLOBAL",1)
-		Global("AerieRomanceActive","GLOBAL",2)
-	Global("ttSimeAerieRomanceBreak","GLOBAL",0)
-THEN
-	RESPONSE #100
-		SetGlobalTimer("ttSimeAerieRomanceBreakTimer","GLOBAL",TWO_DAYS)
-		SetGlobal("ttSimeAerieRomanceBreak","GLOBAL",1)
-END
-
-IF
-	GlobalTimerExpired("ttSimeAerieRomanceBreakTimer","GLOBAL")
-	InParty(Myself)
-	InParty("Aerie")
-	See(Player1)
-	See("Aerie")
-	!StateCheck(Player1,CD_STATE_NOTVALID)
-	!StateCheck("Aerie",CD_STATE_NOTVALID)
-	!See([ENEMY])
-	CombatCounter(0)
-	Global("ttSimeRomanceActive","GLOBAL",2)
-	OR(2)
-		Global("AerieRomanceActive","GLOBAL",1)
-		Global("AerieRomanceActive","GLOBAL",2)
-	Global("ttSimeAerieRomanceBreak","GLOBAL",1)
-THEN
-	RESPONSE #100
-		SetGlobal("ttSimeAerieRomanceBreak","GLOBAL",2)
-END
-
-IF
-	GlobalTimerExpired("ttSimeAerieRomanceBreakTimer","GLOBAL")
-	InParty(Myself)
-	InParty("Aerie")
-	See("Aerie")
-	!StateCheck("Aerie",CD_STATE_NOTVALID)
-	!See([ENEMY])
-	CombatCounter(0)
-	Global("ttSimeRomanceActive","GLOBAL",2)
-	Global("AerieRomanceActive","GLOBAL",3)
-	Global("ttSimeAerieRomanceBreak","GLOBAL",3)
-THEN
-	RESPONSE #100
-		SetGlobal("ttSimeAerieRomanceBreak","GLOBAL",4)
-END
-
-IF
-	Global("ttSimeAerieRomanceBreak","GLOBAL",2)
-	Global("ttSimeAerieRomanceBreak","GLOBAL",4)
-THEN
-	RESPONSE #100
-		StartDialogueNoSet("Aerie")
-END
 
 ///////////////////////////////
 // Jaheira Romance Conflict
 ///////////////////////////////
 
-IF
-	InParty(Myself)
-	InParty("Jaheira")
-	See(Player1)
-	See("Jaheira")
-	!StateCheck(Player1,CD_STATE_NOTVALID)
-	!StateCheck("Jaheira",CD_STATE_NOTVALID)
-	!See([ENEMY])
-	CombatCounter(0)
-	Global("ttSimeRomanceActive","GLOBAL",2)
-	OR(2)
-		Global("JaheiraRomanceActive","GLOBAL",1)
-		Global("JaheiraRomanceActive","GLOBAL",2)
-	Global("ttSimeJaheiraRomanceBreak","GLOBAL",0)
-THEN
-	RESPONSE #100
-		SetGlobalTimer("ttSimeJaheiraRomanceBreakTimer","GLOBAL",TWO_DAYS)
-		SetGlobal("ttSimeJaheiraRomanceBreak","GLOBAL",1)
-END
 
-IF
-	GlobalTimerExpired("ttSimeJaheiraRomanceBreakTimer","GLOBAL")
-	InParty(Myself)
-	InParty("Jaheira")
-	See(Player1)
-	See("Jaheira")
-	!StateCheck(Player1,CD_STATE_NOTVALID)
-	!StateCheck("Jaheira",CD_STATE_NOTVALID)
-	!See([ENEMY])
-	CombatCounter(0)
-	Global("ttSimeRomanceActive","GLOBAL",2)
-	OR(2)
-		Global("JaheiraRomanceActive","GLOBAL",1)
-		Global("JaheiraRomanceActive","GLOBAL",2)
-	Global("ttSimeJaheiraRomanceBreak","GLOBAL",1)
-THEN
-	RESPONSE #100
-		SetGlobal("ttSimeJaheiraRomanceBreak","GLOBAL",2)
-END
-
-IF
-	Global("ttSimeJaheiraRomanceBreak","GLOBAL",2)
-THEN
-	RESPONSE #100
-		StartDialogueNoSet("Jaheira")
-END
 
 ///////////////////////////////
 // Viconia Romance Conflict
 ///////////////////////////////
 
-IF
-	InParty(Myself)
-	InParty("Viconia")
-	See(Player1)
-	See("Viconia")
-	!StateCheck(Player1,CD_STATE_NOTVALID)
-	!StateCheck("Viconia",CD_STATE_NOTVALID)
-	!See([ENEMY])
-	CombatCounter(0)
-	Global("ttSimeRomanceActive","GLOBAL",2)
-	OR(2)
-		Global("ViconiaRomanceActive","GLOBAL",1)
-		Global("ViconiaRomanceActive","GLOBAL",2)
-	Global("ttSimeViconiaRomanceBreak","GLOBAL",0)
-THEN
-	RESPONSE #100
-		SetGlobalTimer("ttSimeViconiaRomanceBreakTimer","GLOBAL",TWO_DAYS)
-		SetGlobal("ttSimeViconiaRomanceBreak","GLOBAL",1)
-END
-
 //Continues in ttsimed.baf
 
 /////////////////////////
