diff --git a/DIALOGUES/YveJNAT.d b/DIALOGUES/YveJNAT.d
index 3399180..7cdb08c 100644
--- Yvette/DIALOGUES/YveJNAT.d
+++ Yvette/DIALOGUES/YveJNAT.d
@@ -146,35 +146,6 @@ END
 
 ///////Viconia Conflict
 
-IF~InParty(Myself) InParty("Viconia") Global("YvetteViconia","GLOBAL",0)~THEN BEGIN Yve0Vx00
-SAY @45 /* This drow - she's a priestess of Shar! I'm sorry, but I can't travel with such a creature. I understand you may not know of it, but Sune and Shar are the greatest of enemies. She--this drow--is probably like her goddess: vicious, treacherous and unpredictable. */
-IF~~THEN REPLY @46 /* Please, don't go. Eh, I knew this would cause trouble. Viconia, you should leave. */ GOTO Yve0Vx01
-IF~~THEN REPLY @47 /* Leave, then. I know Viconia quite well; I've traveled with her before. I trust her, and if you don't, that's your problem. */ GOTO Yve0Vx02
-IF~~THEN REPLY @48 /* Is there really no way to compromise? */ GOTO Yve0Vx03
-END
-
-IF~~THEN BEGIN Yve0Vx03
-SAY @49 /* I'm sorry, but no. What's more I think I can offer you the same help she would. */
-IF~~THEN REPLY @50 /* I knew it would cause trouble. Viconia, you must leave. */ GOTO Yve0Vx01
-IF~~THEN REPLY @51 /* I know Viconia quite well; I've traveled with her before. I trust her. If you don't, that's your problem. */ GOTO Yve0Vx02
-END
-
-IF~~THEN BEGIN Yve0Vx01
-SAY @52 /* Thank you, <CHARNAME>. I'm really grateful. I'm sure she'll find a way to survive on her own. */
-IF~~THEN DO ~SetGlobal("YvetteViconia","GLOBAL",1) ActionOverride("Viconia",LeaveParty())~ EXIT
-END
-
-IF~~THEN BEGIN Yve0Vx02
-SAY @53 /* I'll leave then. Farewell. It's a pity we couldn't travel a bit longer. */
-IF~~THEN DO ~SetGlobal("YvetteViconia","GLOBAL",2) LeaveParty() EscapeArea()~ EXIT
-END
-
-//2nd time
-
-IF ~InParty(Myself) InParty("Viconia") Global("YvetteViconia","GLOBAL",1) !Global("YvetteRomanceActive","GLOBAL",5)~ THEN BEGIN Yve0Vx04
-SAY @54 /* I'm sorry, but I told you how I felt about Viconia and traveling with her. But it would seem you've made your decision. Fare thee well. I wish you luck on the road. */
-IF~~THEN DO ~SetGlobal("YvetteViconia","GLOBAL",2) SetGlobal("YvetteRomanceActive","GLOBAL",3) LeaveParty() EscapeArea()~ EXIT
-END
 
 /////////////////////////////////Romance
 //////////////////1
diff --git a/DIALOGUES/YveNAT.d b/DIALOGUES/YveNAT.d
index 9a29f0f..f25f514 100644
--- Yvette/DIALOGUES/YveNAT.d
+++ Yvette/DIALOGUES/YveNAT.d
@@ -37,7 +37,7 @@ END
 
 IF~~THEN BEGIN Yvette006
 SAY @15 /* Lovely! I'm glad I can stay, dear lord. Come then! I shall support you with my prayers. */
-IF~~THEN DO ~SetGlobal("YvetteJoin","GLOBAL",1)  TakePartyItem("YxDraw") DestroyItem("YxDraw") JoinParty()~ EXIT
+IF~~THEN DO ~SetGlobal("YvetteJoin","GLOBAL",1)  TakePartyItem("YxDraw") DestroyItem("YxDraw") SetGlobal("YvetteWasInParty","GLOBAL",1) JoinParty()~ EXIT
 END
 
 IF~~THEN BEGIN Yvette003
@@ -120,6 +120,7 @@ END
 IF~Global("YvetteJoin","GLOBAL",1) !Global("YvetteRomanceActive","GLOBAL",4)~THEN BEGIN YvetteKickOut000
 SAY @32 /* So, you want me to leave? I'd prefer to stay with you, good lord, but if I must...I...I will leave, then, and search for my home... */
 IF~~THEN REPLY @33 /* Yes, you have to leave. Maybe we'll return for you later. */ GOTO YvetteKickOut001
+IF~~THEN REPLY @300 /* Please, stay here. We'll return for you later. */ GOTO YvetteKickOut001x
 IF~~THEN REPLY @34 /* That was a mistake. Of course you should stay with us! */ GOTO YvetteKickOut002
 END
 
@@ -128,7 +129,22 @@ SAY @35 /* Sorry, but I'll try wandering on my own. Fare thee well, my lord. */
 IF~~THEN DO ~SetGlobal("YvetteRomanceActive","GLOBAL",3) SetGlobal("YvetteJoin","GLOBAL",0) EscapeArea()~ EXIT
 END 
 
+IF~~THEN BEGIN YvetteKickOut001x
+SAY @301 /* It's so good to hear that. Please be quick! */
+IF~~THEN DO ~SetGlobal("YvetteJoin","GLOBAL",0)~ EXIT
+END 
+
 IF~~THEN BEGIN YvetteKickOut002
 SAY @36 /* It's so good to hear that. Let's go, then! */
 IF~~THEN DO ~JoinParty()~ EXIT
 END
+
+IF~Global("YvetteWasInParty","GLOBAL",1) !InParty(Myself)~ YvetteCameSeeAgain
+SAY @302 /* You're back= Shall we go? */
+IF~~THEN REPLY @303 /* Yes, let's go. */ GOTO YvetteKickOut002
+IF~~THEN REPLY @304 /* Not yet, please wait a little more. */ GOTO YvetteWaitMore
+END
+
+IF~~THEN BEGIN YvetteWaitMore
+SAY @305 /* Aww... Ok, Please be quick! */
+END 
diff --git a/DIALOGUES/YxYve.d b/DIALOGUES/YxYve.d
index 0ad8ccb..58267fa 100644
--- Yvette/DIALOGUES/YxYve.d
+++ Yvette/DIALOGUES/YxYve.d
@@ -128,6 +128,7 @@ END
 IF~Global("YvetteJoin","GLOBAL",1) !Global("YvetteRomanceActive","GLOBAL",4)~THEN BEGIN YvetteKickOut000
 SAY @33 /* So, you want me to leave? I'd prefer to stay with you, good lord, but if I must...I...I will leave, then, and serch for my home... */
 IF~~THEN REPLY @34 /* Yes, you have to leave. Maybe we'll return for you later. */ GOTO YvetteKickOut001
+IF~~THEN REPLY @300 /* Please, stay here. We'll return for you later. */ GOTO YvetteKickOut001x
 IF~~THEN REPLY @35 /* That was a mistake. Of course you should stay with us! */ GOTO YvetteKickOut002
 END
 
@@ -136,7 +137,22 @@ SAY @36 /* Sorry, but I'll try wandering on my own. Fare thee well, my lord. */
 IF~~THEN DO ~SetGlobal("YvetteRomanceActive","GLOBAL",3) SetGlobal("YvetteJoin","GLOBAL",0) EscapeArea()~ EXIT
 END 
 
+IF~~THEN BEGIN YvetteKickOut001x
+SAY @301 /* It's so good to hear that. Please be quick! */
+IF~~THEN DO ~SetGlobal("YvetteJoin","GLOBAL",0)~ EXIT
+END 
+
 IF~~THEN BEGIN YvetteKickOut002
 SAY @37 /* It's so good to hear that. Let's go, then! */
 IF~~THEN DO ~JoinParty()~ EXIT
-END
+END
+
+IF~Global("YvetteWasInParty","GLOBAL",1) !InParty(Myself)~ YvetteCameSeeAgain
+SAY @302 /* You're back! Shall we go? */
+IF~~THEN REPLY @303 /* Yes, let's go. */ GOTO YvetteKickOut002
+IF~~THEN REPLY @304 /* Not yet, please wait a little more. */ GOTO YvetteWaitMore
+END
+
+IF~~THEN BEGIN YvetteWaitMore
+SAY @305 /* Aww... Ok, Please be quick! */
+END 
diff --git a/SCRIPTS/Player.baf b/SCRIPTS/Player.baf
index 3e4de52..432cd71 100644
--- Yvette/SCRIPTS/Player.baf
+++ Yvette/SCRIPTS/Player.baf
@@ -36,14 +36,6 @@ END
 
 ////////////Viconia and Yvette
 
-IF
-Global("YvetteViconia","GLOBAL",2)
-THEN
-RESPONSE #100
-SetGlobal("YvetteViconia","GLOBAL",3)
-SetGlobal("YvetteRomanceActive","GLOBAL",3)
-END
-
 /////////////Yvette spellhold
 
 IF
diff --git a/SCRIPTS/YxYve.baf b/SCRIPTS/YxYve.baf
index 104edb6..51a56c7 100644
--- Yvette/SCRIPTS/YxYve.baf
+++ Yvette/SCRIPTS/YxYve.baf
@@ -347,24 +347,6 @@ END
 
 //////////////Viconia Conflict
 
-IF
-InParty(Myself)
-InParty("Viconia")
-Global("YvetteViconia","GLOBAL",0)
-THEN
-RESPONSE #100
-StartDialog("YxYveJ",Player1)
-END
-
-IF
-InParty(Myself)
-InParty("Viconia")
-Global("YvetteViconia","GLOBAL",1)
-THEN
-RESPONSE #100
-StartDialog("YxYveJ",Player1)
-END
-
 ///////////////Lathander
 
 IF
diff --git a/Translations/english/YXYVE.TRA b/Translations/english/YXYVE.TRA
index 0eb6039..e64eeac 100644
--- Yvette/Translations/english/YXYVE.TRA
+++ Yvette/Translations/english/YXYVE.TRA
@@ -36,3 +36,10 @@
 @35   = ~That was a mistake. Of course you should stay with us!~
 @36   = ~Sorry, but I'll try wandering on my own. Fare thee well, my lord.~
 @37   = ~It's so good to hear that. Let's go, then!~
+
+@300  = ~Please, stay here. We'll return for you later.~
+@301  = ~It's so good to hear that. Please be quick!~
+@302  = ~You're back! Shall we go?~
+@303  = ~Yes, let's go.~
+@304  = ~Not yet, please wait a little more.~
+@305  = ~Aww... Ok, Please be quick!~
